<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="João P. N. Silva" />
  <meta name="author" content="Soraia Vidigal Lopes" />
  <meta name="author" content="Diogo J. Grilo" />
  <meta name="author" content="Zach Hensel" />
  <meta name="dcterms.date" content="2019-05-07" />
  <meta name="keywords" content="gene expression, synthetic biology, transcription" />
  <title>Plasmids for independently tunable, low-noise expression of two genes</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Plasmids for independently tunable, low-noise expression of two genes</h1>
</header>
<p><em>A DOI-citable version of this manuscript is available at <a href="https://doi.org/10.1101/515940" class="uri">https://doi.org/10.1101/515940</a></em>.</p>
<p><small><em> This manuscript was automatically generated on May 7, 2019. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>João P. N. Silva</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0003-4455-4286">0000-0003-4455-4286</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Soraia Vidigal Lopes</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0002-3705-2619">0000-0002-3705-2619</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Diogo J. Grilo</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0003-1279-3629">0000-0003-1279-3629</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Zach Hensel</strong><br> <img src="images/orcid.svg" alt="ORCID icon" class="inline_icon" /> <a href="https://orcid.org/0000-0002-4348-6229">0000-0002-4348-6229</a> · <img src="images/github.svg" alt="GitHub icon" class="inline_icon" /> <a href="https://github.com/zach-hensel">zach-hensel</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal · Funded by Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) <br>Correspondence: <a href="mailto:zach.hensel@itqb.unl.pt">zach.hensel@itqb.unl.pt</a> </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Some microbiology experiments and biotechnology applications can be improved if it is possible to tune the expression of two different genes at the same time with cell-to-cell variation at or below the level of genes constitutively expressed from the chromosome (the “extrinsic noise limit”). This was recently achieved for a single gene by exploiting negative autoregulation by the tetracycline repressor (TetR) and bicistronic gene expression to reduce gene expression noise. We report new plasmids that use the same principles to achieve simultaneous, low-noise expression for two genes in <em>Escherichia coli</em>. The TetR system was moved to a compatible plasmid backbone, and a system based on the <em>lac</em> repressor (LacI) was found to also exhibit gene expression noise below the extrinsic noise limit. We characterized gene expression mean and noise across the range of induction levels for these plasmids, applied the LacI system to tune expression for single-molecule mRNA detection in two different growth conditions, and showed that two plasmids can be co-transformed to independently tune expression of two different genes.</p>
<h2 id="introduction" class="page_break_before">Introduction</h2>
<p>We recently reported the development of a plasmid-based gene-expression system in which a gene of interest was expressed bicistronically with the tetracycline repressor (TetR) <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. Using this gene expression system, cell-to-cell variation was below the “extrinsic noise limit” (coefficient of variation squared of protein concentration, <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?CV%5E2%20%5Capprox%200.1" alt="CV^2 \approx 0.1" title="CV^2 \approx 0.1" />) observed for genes expressed from the chromosome <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>. When TetR and GFP were expressed bicistronically, GFP induction and gene expression noise were similar to those observed for a TetR-GFP fusion protein with autoregulation <span class="citation" data-cites="GcRoYBF2">[<a href="#ref-GcRoYBF2">3</a>]</span>. Compared to induction of gene expression under the control of a constitutively expressed transcriptional repressor, the inducer dose-response was relatively linearized, and gene expression noise was much lower at intermediate induction levels <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>.</p>
<p>Our experiments in mRNA detection and other single-molecule applications in living <em>E. coli</em> cells sometimes require the tunable expression of two different genes, both with low noise levels. For example, adopting a recently reported mRNA detection system based on local enrichment of fluorescent RNA-binding proteins <span class="citation" data-cites="82diXnad">[<a href="#ref-82diXnad">4</a>]</span> for use in <em>E. coli</em> requires lower noise in protein production relative to the same system in <em>S. cerevisiae</em> because of a smaller cell volume for <em>E. coli</em> and the inability to sequester unbound protein in the nucleus. At the same time, tunable expression with low noise in the level of the target RNA is desired to make it possible to characterize the accuracy of RNA detection over a range of RNA levels. We hoped that expressing both the target RNA and RNA-binding fluorescent protein on two plasmids that could be tuned independently would simplify and accelerate development of new RNA-detection systems in <em>E. coli</em>. Achieving this was a three-step process: first, characterizing the TetR-based system on a compatible plasmid backbone; second, establishing an orthogonal, low-noise expression system based on the <em>lac</em> repressor (LacI); and third, showing that the two systems can be tuned independently.</p>
<h2 id="results">Results</h2>
<h3 id="moving-bicistronic-autoregulatory-construct-to-a-compatible-plasmid-backbone">Moving bicistronic autoregulatory construct to a compatible plasmid backbone</h3>
<p>The first step in creating a low-noise system for tuning expression of two genes was to establish that a previously characterized, bicistronic autoregulatory circuit functions well in a compatible plasmid backbone. In this expression system, GFP and TetR are expressed bicistronically from the TetR-repressible promoter P<sub>LtetO-1</sub> and expression is induced by the addition of ATc <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. This system was shown to have low noise and a linearized dose response compared to a system in which TetR was constitutively expressed. We moved the system from a plasmid with a p15A replicon conferring ampicillin resistance to a lower-copy-number plasmid with a pSC101 replicon conferring spectinomycin resistance <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">5</a>]</span>. The p15A and pSC101 replicons have been used together in multiplasmid systems <span class="citation" data-cites="15TQ1MvYV">[<a href="#ref-15TQ1MvYV">6</a>]</span>.</p>
<p>GFP expression mean and noise were characterized from low to high levels of induction by flow cytometry. <strong>Figure <a href="#fig:pZH509-pJS101">1</a></strong> shows that pJS101 induces at similar ATc concentrations as pZH509, with the change to the lower-copy pSC101 backbone resulting in a 58% drop in mean expression levels at a wide range of ATc concentrations. For a similar expression system in the absence of autoregulated TetR expression, moving the P<sub>LtetO-1</sub> promoter from a p15A to a pSC101 backbone resulted in an 87% drop in expression <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">7</a>]</span>. A smaller change was expected in our experiment, since negative autoregulation provided dosage compensation, just as autoregulation can reduce noise in plasmid copy number <span class="citation" data-cites="GcRoYBF2 Qz6zNiXK S7KRThVS">[<a href="#ref-GcRoYBF2">3</a>,<a href="#ref-Qz6zNiXK">8</a>,<a href="#ref-S7KRThVS">9</a>]</span>.</p>
<figure>
<img src="images/Fig1_pZH509_pJS101.png" alt="Figure 1: Moving the TetR expression system to a compatible plasmid backbone. Cultures of E. coli MG1655 harboring pZH509 (p15a origin, blue) or pJS101 (pSC101 origin, orange) expressing GFPmut2 with bicistronic autoregulation by TetR were grown at 30 °C in rich media with induction of 1, 5, 25 and 125 nM ATc. Mean single-cell GFP fluorescence was estimated using flow cytometry. Mean GFP levels in 3 independent replicates are indicated with different shapes. Black lines indicate the mean plus or minus 1 standard error of the mean from the 3 replicates." id="fig:pZH509-pJS101" style="width:89mm" /><figcaption><span>Figure 1:</span> <strong>Moving the TetR expression system to a compatible plasmid backbone.</strong> Cultures of <em>E. coli</em> MG1655 harboring pZH509 (p15a origin, blue) or pJS101 (pSC101 origin, orange) expressing GFPmut2 with bicistronic autoregulation by TetR were grown at 30 °C in rich media with induction of 1, 5, 25 and 125 nM ATc. Mean single-cell GFP fluorescence was estimated using flow cytometry. Mean GFP levels in 3 independent replicates are indicated with different shapes. Black lines indicate the mean plus or minus 1 standard error of the mean from the 3 replicates.</figcaption>
</figure>
<h3 id="an-alternative-regulatory-construct-with-laci-replacing-tetr">An alternative regulatory construct with LacI replacing TetR</h3>
<p>We hypothesized that replacing P<sub>LtetO-1</sub> with the IPTG-inducible promoter P<sub>LlacO-1</sub>, with similar characteristics <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">7</a>]</span>, and replacing TetR with LacI might result in a similarly useful expression system that could be tuned independently. However, regulatory parameters for TetR and LacI vary significantly. TetR binds <em>tetO2</em> more strongly than LacI binds <em>lacO1</em> (approximately 0.5—1.0 kcal/mol difference in binding energy <span class="citation" data-cites="IbIaRemg UMSODjob">[<a href="#ref-IbIaRemg">10</a>,<a href="#ref-UMSODjob">11</a>]</span> for a single site, with 2 tandem sites in our constructs). And, TetR binds ATc much more strongly than LacI binds IPTG (over 3 orders of magnitude difference in typical concentrations required for half-induction <span class="citation" data-cites="Ls5d7x4o WxFVKQqb">[<a href="#ref-Ls5d7x4o">12</a>,<a href="#ref-WxFVKQqb">13</a>]</span>).</p>
<p>We first characterized induction of GFP expression in MG1655 cells harboring IPTG-inducible pJS102 by flow cytometry. <strong>Figure <a href="#fig:pJS102-mean-noise">2</a>a</strong> shows an induction range of almost 2 orders of magnitude from 0 to 1250 µM IPTG, with very good reproducibility of induction levels in 3 independent experiments. Previous experiments with the TetR-based system showed a similar total dynamic range, but with a large jump in expression going from 0 nM to 0.5 nM ATc <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. This effect is not seen for pJS102, suggesting that switching from TetR:ATc to LacI:IPTG improves the dynamic range of tunable induction levels to a small extent.</p>
<figure>
<img src="images/Fig2_pJS102_mean_noise.png" alt="Figure 2: Characterizing mean expression levels and noise for different gene expression systems. Cultures of E. coli MG1655 harboring pJS102 (LacI regulation, p15a origin, green), pZH509 (TetR regulation, p15a origin, blue) or pJS101 (TetR regulation, pSC101 origin, orange) were grown at 30 °C in rich media with induction by IPTG (0, 2, 4.5, 10, 22.4, 50, 111.8, 250, 559 and 1250 µM) or ATc (1, 5, 25 and 125 nM). Mean single-cell GFP fluorescence and GFP expression noise was estimated using flow cytometry. Data from 3 independent replicates are indicated with different shapes. Black lines indicate the mean GFP expression and expression noise plus or minus 1 standard error of the mean from the 3 replicates. (a) Mean GFP expression for pJS102 at different IPTG concentrations shows tunable and reproducible induction over an ~50-fold dynamic range. Expression at zero IPTG is plotted separately to fit on logarithmic scale. (b) GFP expression noise as a function of mean for pZH509, pJS101 and pJS102. For all samples, GFP mean increases monotonically with inducer concentration. Note that the full range of induction is not shown here for pZH509 and pJS101 as even 1 nM ATc induces expression. GFP expression noise is low in all conditions for all strains." id="fig:pJS102-mean-noise" style="width:183mm" /><figcaption><span>Figure 2:</span> <strong>Characterizing mean expression levels and noise for different gene expression systems.</strong> Cultures of <em>E. coli</em> MG1655 harboring pJS102 (LacI regulation, p15a origin, green), pZH509 (TetR regulation, p15a origin, blue) or pJS101 (TetR regulation, pSC101 origin, orange) were grown at 30 °C in rich media with induction by IPTG (0, 2, 4.5, 10, 22.4, 50, 111.8, 250, 559 and 1250 µM) or ATc (1, 5, 25 and 125 nM). Mean single-cell GFP fluorescence and GFP expression noise was estimated using flow cytometry. Data from 3 independent replicates are indicated with different shapes. Black lines indicate the mean GFP expression and expression noise plus or minus 1 standard error of the mean from the 3 replicates. (<strong>a</strong>) Mean GFP expression for pJS102 at different IPTG concentrations shows tunable and reproducible induction over an ~50-fold dynamic range. Expression at zero IPTG is plotted separately to fit on logarithmic scale. (<strong>b</strong>) GFP expression noise as a function of mean for pZH509, pJS101 and pJS102. For all samples, GFP mean increases monotonically with inducer concentration. Note that the full range of induction is not shown here for pZH509 and pJS101 as even 1 nM ATc induces expression. GFP expression noise is low in all conditions for all strains.</figcaption>
</figure>
<p>Next, we compared noise in protein expression, with the concern that the <em>lac</em> operon present in the MG1655 host strain could lead to all-or-none expression at intermediate IPTG concentrations <span class="citation" data-cites="eUtnfAD6">[<a href="#ref-eUtnfAD6">14</a>]</span>. However, <strong>Figure <a href="#fig:pJS102-mean-noise">2</a>b</strong> shows low noise in GFP expression at all IPTG concentrations, with noise levels comparable to pZH509 and pJS102 at the same mean GFP levels. Note that apparently high noise at very low expression was partially due to measurement noise, and, at any rate, was much lower than noise when expression is regulated by a constitutively expressed repressor <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>.</p>
<p>We found that side scattering was weakly correlated with fluorescence, and thus with cell size, so gating for scattering modestly reduced measured noise in fluorescence intensity. However, we are comparing to an “extrinsic noise limit” determined from measurements of cell fluorescence divided by cell area <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>, which effectively does the same thing. In practice, we observed slightly lower noise measurements for GFP concentration in fluorescence microscopy images compared to total GFP fluorescence in the gated flow cytometry sample for similarly induced strains. This difference was more significant for very-low-expression conditions, and noise in conditions where GFP fluorescence distributions significantly overlapped with ungated background events (GFP intensity less than 10<sup>4</sup> in <strong>Figure <a href="#fig:gammaFits">S1</a></strong>) was somewhat overestimated. Our noise measurements were also consistent with the lower limit of gene expression noise found in many <em>E. coli</em> promoters using a similar flow cytometry method with similar gating and fitting procedures <span class="citation" data-cites="16n3gU60l">[<a href="#ref-16n3gU60l">15</a>]</span>.</p>
<h3 id="using-the-new-induction-system-for-detection-of-single-mrna-in-living-e.-coli">Using the new induction system for detection of single mRNA in living <em>E. coli</em></h3>
<p>Recently, an improved method for detection of mRNA by local enrichment of fluorescent RNA-binding proteins was reported in <em>S. cerevisiae</em> <span class="citation" data-cites="82diXnad">[<a href="#ref-82diXnad">4</a>]</span>. This reduced the aggregation of mRNAs bound by the bacteriophage MS2 coat protein, which has also caused mRNA immortalization that has limited experiments in <em>E. coli</em> to observing transcription just after induction <span class="citation" data-cites="TvXapFFQ">[<a href="#ref-TvXapFFQ">16</a>]</span>. We hypothesized that aggregation could be reduced by reducing the expression levels of both mRNA and mRNA-binding proteins, and by having low cell-to-cell variation in expression. We developed a strain in which mRNA molecules encode mVenus-Cro and also include 24 tandem repeats of the binding sequence for the PP7 coat protein (PP7cp) <span class="citation" data-cites="1GFLV3s4q">[<a href="#ref-1GFLV3s4q">17</a>]</span>. These mRNAs were constitutively expressed at low levels (less than 1 molecule per cell). Plasmid pJS102 was used as a template to develop a fluorescent, IPTG-inducible reporter of expression, PP7cp-SYFP2.</p>
<p>We tested the utility of this expression system for tuning low-noise gene expression in different growth conditions. In previous experiments, we expressed the RNA-binding protein from a constitutive promoter integrated into the chromosome; this required long cycles of optimization every time a parameter was changed (e.g. growth media, temperature, and fluorescent protein sequence) that changed protein expression levels. <strong>Figure <a href="#fig:pZH713-optimization">3</a>a</strong> shows that single-molecule mRNA detection was optimal (fluorescent mRNA spots are sufficiently bright but not obscured by background) at 100 µM IPTG in minimal media supplemented with 1% rich media. We note the absence of pole-localized fluorescent spots that characterize mRNA aggregation <span class="citation" data-cites="uQeM79z">[<a href="#ref-uQeM79z">18</a>]</span>, and we observed reasonable mRNA lifetimes of a few minutes in time-lapse imaging.</p>
<p>We moved to minimal media to explore a growth condition with different mRNA expression levels and slower growth rates, but found that 100 µM IPTG gave a background of unbound PP7cp-SYFP2 molecules that often made it impossible to detect mRNA spots. <strong>Figure <a href="#fig:pZH713-optimization">3</a>b</strong> shows how the IPTG-inducible expression system made it simple to quickly scan different PP7cp-SYFP2 induction conditions and identify 10–20 µM IPTG as a range in which PP7cp-SYFP2 levels were high enough to label single mRNAs, but not so high as to give a high background of unbound molecules. Lastly, we note that the strain used for mRNA imaging has its entire <em>lac</em> operon replaced by the synthetic construct. Thus, this expression system works well both in the presence and absence of the <em>lac</em> operon.</p>
<figure>
<img src="images/Fig3_pZH713_optimization.png" alt="Figure 3: Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions. ZHX99 cells were grown with low expression levels (less than 1 molecule per cell) of mRNAs harboring 24 tandem binding sites for the PP7 coat protein fused to SYFP2. Cells were spotted on agarose gel pads and YFP fluorescence images were acquired. Two example images are shown for each condition. Images were taken shortly after preparing samples, so adjacent cells were not usually closely related in cell lineages. (a) PP7cp-SYFP2 was induced with 100 µM IPTG to detect single mVenus-Cro mRNA molecules in supplemented media and minimal media conditions; in minimal media there was too high a PP7cp-SYFP expression level to see single mRNA spots above background. Scale bar 4 µm. (b) Using the pJS102 expression system, PP7cp-SYFP2 expression levels were varied by induction with 5, 10, 20 and 40 µM IPTG. The range of 10—20 µM IPTG was identified to give bright mRNA spots above the background of unbound PP7cp-SYFP2 molecules. Scale bar 2 µm." id="fig:pZH713-optimization" style="width:89mm" /><figcaption><span>Figure 3:</span> <strong>Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.</strong> ZHX99 cells were grown with low expression levels (less than 1 molecule per cell) of mRNAs harboring 24 tandem binding sites for the PP7 coat protein fused to SYFP2. Cells were spotted on agarose gel pads and YFP fluorescence images were acquired. Two example images are shown for each condition. Images were taken shortly after preparing samples, so adjacent cells were not usually closely related in cell lineages. (<strong>a</strong>) PP7cp-SYFP2 was induced with 100 µM IPTG to detect single mVenus-Cro mRNA molecules in supplemented media and minimal media conditions; in minimal media there was too high a PP7cp-SYFP expression level to see single mRNA spots above background. Scale bar 4 µm. (<strong>b</strong>) Using the pJS102 expression system, PP7cp-SYFP2 expression levels were varied by induction with 5, 10, 20 and 40 µM IPTG. The range of 10—20 µM IPTG was identified to give bright mRNA spots above the background of unbound PP7cp-SYFP2 molecules. Scale bar 2 µm.</figcaption>
</figure>
<h3 id="independent-tunable-expression-of-two-genes">Independent, tunable expression of two genes</h3>
<p>Lastly, we tested whether ATc-inducible and IPTG-inducible plasmids could be combined to achieve low-noise expression of two genes in the same cell. We replaced GFPmut2 in pJS102 with the fast-maturing RFP mScarlet-I <span class="citation" data-cites="zj8Ydkzo">[<a href="#ref-zj8Ydkzo">19</a>]</span> to create the plasmid pDG101. This plasmid was co-transformed with pJS101 into <em>E. coli</em> MG1655, and green and red fluorescence were compared at different combinations of ATc and IPTG concentrations. <strong>Figure <a href="#fig:pJS101-pDG101">4</a>a</strong> shows that pJS101 induction by ATc was unaffected by pDG101 induction by IPTG, and that all conditions gave low noise in GFP concentration. <strong>Figure <a href="#fig:pJS101-pDG101">4</a>b</strong> shows that mScarlet-I expression from pDG101 was similarly unaffected by the level of pJS101 induction by ATc. <strong>Figure <a href="#fig:pJS101-pDG101">4</a>c</strong> and <strong><a href="#fig:pJS101-pDG101">4</a>d</strong> shows that mean expression levels were reproducible in 3 independent replicates, with low noise in each sample across a 5-fold increase in expression levels. There is some day-to-day variation in mean expression levels, but for each replicate induction of GFPmut2 and mScarlet-I is independent. Thus, independent, tunable expression of two genes can be achieved by replacing GFPmut2 in pJS101 and pJS102 with other genes of interest and co-transforming the plasmids into <em>E. coli</em>.</p>
<figure>
<img src="images/Fig4_pJS101_pDG101.png" alt="Figure 4: Independence of induction of TetR and LacI systems. MG1655 cells harboring pJS101 (ATc-inducible expression of GFP, pSC101 origin) and pDG101 (IPTG-inducible expression of mScarlet-I, p15a origin) were grown with different combinations of IPTG and ATc concentrations. GFP and mScarlet-I fluorescence was observed and quantified by fluorescence microscopy. (a) GFP fluorescence showed no apparent influence of IPTG on ATc-induced expression of GFP for cells grown at 30 °C in EZ-RICH media. (b) For the same cells as in a, no influence of ATc was observed on IPTG-induced expression of mScarlet-I. Scale bar 3 µm. (c,d) Cells were grown at 37 °C in M9A media in 3 independent replicates, and the mean and noise of GFP and mScarlet-I fluorescence intensities (from the average intensities of pixels within cells) were estimated at different IPTG (50, 1250 µM) and ATc (5, 125 nM) concentrations. Translation was inhibited with chloramphenicol for 1 hour prior to preparing microscope samples in order to allow for fluorescent protein maturation. Colored circles indicate the results from independent replicates. For mean expression levels in c, black lines indicate the mean GFP and mScarlet-I expression plus or minus 1 standard error of the mean from the 3 replicates. Noise measurements in d indicate low noise in both channels in all conditions and replicates." id="fig:pJS101-pDG101" style="width:89mm" /><figcaption><span>Figure 4:</span> <strong>Independence of induction of TetR and LacI systems.</strong> MG1655 cells harboring pJS101 (ATc-inducible expression of GFP, pSC101 origin) and pDG101 (IPTG-inducible expression of mScarlet-I, p15a origin) were grown with different combinations of IPTG and ATc concentrations. GFP and mScarlet-I fluorescence was observed and quantified by fluorescence microscopy. (<strong>a</strong>) GFP fluorescence showed no apparent influence of IPTG on ATc-induced expression of GFP for cells grown at 30 °C in EZ-RICH media. (<strong>b</strong>) For the same cells as in <strong>a</strong>, no influence of ATc was observed on IPTG-induced expression of mScarlet-I. Scale bar 3 µm. (<strong>c</strong>,<strong>d</strong>) Cells were grown at 37 °C in M9A media in 3 independent replicates, and the mean and noise of GFP and mScarlet-I fluorescence intensities (from the average intensities of pixels within cells) were estimated at different IPTG (50, 1250 µM) and ATc (5, 125 nM) concentrations. Translation was inhibited with chloramphenicol for 1 hour prior to preparing microscope samples in order to allow for fluorescent protein maturation. Colored circles indicate the results from independent replicates. For mean expression levels in <strong>c</strong>, black lines indicate the mean GFP and mScarlet-I expression plus or minus 1 standard error of the mean from the 3 replicates. Noise measurements in <strong>d</strong> indicate low noise in both channels in all conditions and replicates.</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<h3 id="implementing-low-noise-expression-systems">Implementing low-noise expression systems</h3>
<p>Using modern molecular cloning techniques, it is simple to replace GFPmut2 in pJS101 and pJS102 with genes of interest by PCR and isothermal assembly with near 100% efficiency and a low probability of clones with incorrect sequence. The apparent insensitivity of this circuit to regulatory parameters such as binding affinities for the repressor to DNA and inducer suggests that it can be easily extended to a third, repressor-based expression system. Further, additional ribosome binding sites can be added to the bicistronic operon to express additional components. Notably, we have found that the ATc- and IPTG-inducible systems both function well in MG1655, a strain in which the <em>lac</em> operon was deleted, and the <em>E. coli</em> TOP10 strain. This host insensitivity could be specific to repressors that are not encoded by the host strain, such as TetR, or repressors that are expressed at very low levels, such as LacI <span class="citation" data-cites="YoMCfbca">[<a href="#ref-YoMCfbca">20</a>]</span>.</p>
<h3 id="functionality-in-other-organisms">Functionality in other organisms</h3>
<p>We chose p15a and pSC101 plasmids for these systems because we primarily apply them in <em>E. coli</em>, they transform efficiently, and they have been successfully co-transformed in earlier work <span class="citation" data-cites="15TQ1MvYV">[<a href="#ref-15TQ1MvYV">6</a>]</span>. However, these are narrow host-range plasmids, and it remains to be seen whether our expression systems will work well in broad host-range plasmids <span class="citation" data-cites="N7fJlkrJ">[<a href="#ref-N7fJlkrJ">21</a>]</span>, in other organisms, or upon chromosome integration. We expect that the system will be reasonably portable in hosts meeting some basic criteria. First, the P<sub>LtetO-1</sub> and P<sub>LlacO-1</sub> promoters are very strong, with sequences close to the <em>E. coli</em> σ<sub>70</sub> consensus; this promoter must match promoter sequences recognized in another system. Second, we have created expression systems using a variety of ribosome binding sites with different translation rates <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>; translation rates can be predicted from homology to 16S rRNA and other factors independent of the host <span class="citation" data-cites="pzTydEMA">[<a href="#ref-pzTydEMA">22</a>]</span>, and may need to be modified to achieve a desired range of induction. Third, we used the strong, Rho-independent rrnB T1 terminator <span class="citation" data-cites="xjKSoV9v">[<a href="#ref-xjKSoV9v">23</a>]</span>, which should work in a broad range of microbial hosts, but may be less effective in some. Lastly, the addition of an insulating transcriptional repressor ahead of the P<sub>LtetO-1</sub> and P<sub>LlacO-1</sub> promoters is likely to reduce sensitivity to transcription upstream of these constructs. We also note that noise for pJS101, with its lower-copy-number pSC101 replicon, is lower than that for pZH509 or pJS102 at similar expression levels (<strong>Figure <a href="#fig:pJS102-mean-noise">2</a>b</strong>). This suggests that incorporating this construct into the chromosome, where copy number is more tightly regulated, may lead to further noise reduction.</p>
<h3 id="possible-applications">Possible applications</h3>
<p>We expect that the expression plasmids introduced here will be useful for diverse applications in molecular biology. Expression and purification of heteromeric protein complexes could be improved by stoichiometric production of their components, mimicking proportional synthesis in natural systems <span class="citation" data-cites="12aHdKu6v">[<a href="#ref-12aHdKu6v">24</a>]</span>. Additionally, low-noise expression can improve protein production yields <span class="citation" data-cites="5dC8qsZ9">[<a href="#ref-5dC8qsZ9">25</a>]</span>. These systems could also be used in synthetic biology applications where yields can be improved by sequential induction of different components with low cell-to-cell variability. The capacity for low-noise expression at very low expression levels makes them particularly promising for single-molecule imaging experiments or for recombinant expression of low-copy-number components with low cell-to-cell variation to reproduce chromosomal expression levels. We see two major drawbacks to our gene expression system. First, the dynamic range of inducible expression is lower than for systems controlled by constitutively expressed transcriptional repressors <span class="citation" data-cites="NCWOk1rp TlAItren">[<a href="#ref-TlAItren">1</a>,<a href="#ref-NCWOk1rp">7</a>]</span>, because some expression must occur at zero inducer concentration before negative feedback kicks in. While the dynamic range could possibly be expanded by increasing repressor binding strength or having a low level of constitutive repressor expression, we have yet to succeed in this. Second, TetR and LacI are expressed at different levels at different induction conditions. This could have off-target effects (<em>e.g.</em> from non-specific DNA binding); we have occasionally observed slow growth at very high induction levels (over 200 nM ATc for pZH509), but we have not tested whether this is due to high TetR expression or high GFPmut2 expression.</p>
<h2 id="materials-and-methods">Materials and methods</h2>
<h3 id="strain-construction">Strain construction</h3>
<p>All plasmids were constructed using isothermal assembly <span class="citation" data-cites="dc6GAnhL">[<a href="#ref-dc6GAnhL">26</a>]</span> of fragments generated by PCR or double-stranded DNA synthesis (IDT, Coralville) and transformed into Top10 E. coli cells (5-1600-020, IBA Life Sciences, Göttingen). Transformants were screened by colony PCR and verified by sequencing (StabVida, Caparica). Purified plasmids were transformed into <em>E. coli</em> strain MG1655 by growing 3 mL of culture in SOB media at 30 °C to an optical density at 600 nm (OD600) of 0.4, washing twice with 1 mL ice-cold water, resuspending in 40 µL water, electroporating 1—10 ng plasmid with the EC1 setting of a Micropulser (Bio-Rad Laboratories, Hercules) and recovering for 1 hour at 37 °C in SOC media before plating on selective LB-agar.</p>
<p>To generate pJS101 with a compatible backbone, plasmid pZH509 <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span> was used as a template to amplify the bicistronic regulatory construct including the P<sub>LtetO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">7</a>]</span>, GFPmut2 <span class="citation" data-cites="13TJF2gcl">[<a href="#ref-13TJF2gcl">27</a>]</span>, tn10 TetR <span class="citation" data-cites="Q1a7tU8">[<a href="#ref-Q1a7tU8">28</a>]</span> and rrnB T1 transcription terminator <span class="citation" data-cites="xjKSoV9v">[<a href="#ref-xjKSoV9v">23</a>]</span>. Using isothermal assembly, this construct was inserted into the pGB2 backbone <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">5</a>]</span> with the pSC101 origin of replication and spectinomycin resistance to generate plasmid pJS101. Plasmid pJS102 was generated by 3-fragment isothermal assembly. Plasmid pZH509 was used as a template both for the vector backbone and for GFPmut2, with non-homologous extensions added to PCR primers to generate the P<sub>LlacO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">7</a>]</span>. LacI <span class="citation" data-cites="YoMCfbca">[<a href="#ref-YoMCfbca">20</a>]</span> was amplified from <em>E. coli</em> MG1655 by colony PCR.</p>
<p>The test strain for mRNA imaging, ZHX99, was constructed similarly to ZHX222 in recent work <span class="citation" data-cites="A1kNMUKB">[<a href="#ref-A1kNMUKB">29</a>]</span>. In ZHX99, a construct in which a fusion protein of mVenus and Cro is expressed from the bacteriophage λ promoter <em>P<sub>R</sub></em> was integrated into the chromosome to replace the <em>lac</em> operon in MG1655 <span class="citation" data-cites="RBKXt78E">[<a href="#ref-RBKXt78E">30</a>]</span>. ZHX99 differs from ZHX222 in three ways. First, the <em>P<sub>R</sub></em> promoter was weakened by site-directed mutagenesis to produce a strain with lower mRNA levels. Second, a very strong ribosome binding site was added (RBS #136 <span class="citation" data-cites="pzTydEMA">[<a href="#ref-pzTydEMA">22</a>]</span>). Lastly, 24 tandem repeats of the recognition sequence for the PP7 coat protein (PP7cp) were inserted between the open reading frame and transcription terminator (amplified by PCR from pDZ251 <span class="citation" data-cites="1GFLV3s4q">[<a href="#ref-1GFLV3s4q">17</a>]</span>). The pZH713 plasmid for mRNA detection was constructed by replacing GFPmut2 in pJS102 with a fusion protein of SYFP2 (amplified from a plasmid <span class="citation" data-cites="XuRR8Zmt">[<a href="#ref-XuRR8Zmt">31</a>]</span>) and PP7cp (generated after codon optimization by DNA synthesis based on previously reported sequences <span class="citation" data-cites="EObeLKJ1">[<a href="#ref-EObeLKJ1">32</a>]</span>). Additionally, in pZH713 the PP7cp-SYFP2 fusion protein is translated from the weak ribosome binding site from pZH511 <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. We note that mVenus expression in ZHX99 is extremely low (undetectable without strong laser excitation) and does not interfere with mRNA detection by localizing up to 48 SYFP2 molecules in a diffraction-limited spot bound to a single mRNA molecule.</p>
<p>To test independent induction of two genes, GFPmut2 in pJS102 was replaced by mScarlet-I (amplified from a plasmid <span class="citation" data-cites="zj8Ydkzo">[<a href="#ref-zj8Ydkzo">19</a>]</span>) to make pDG101. Plasmids were co-transformed into MG1655 by electroporation following the above protocol, except with 1 µL each undiluted plasmid (20—40 ng) and selecting on LB-agar plates with both spectinomycin and carbenicillin. Sequence maps are included in <strong>File S2</strong> and plasmids useful for constructing additional two-gene expression systems (pJS101 and pJS102, <strong>Table <a href="#tbl:plasmidList">1</a></strong>) are available from AddGene (deposit numbers 118280 and 118281) and have been verified by whole-plasmid sequencing <span class="citation" data-cites="q8xLcfYV">[<a href="#ref-q8xLcfYV">33</a>]</span>.</p>
<a name="tbl:plasmidList"></a>
<table>
<caption><span>Table 1:</span> Plasmids used in this study. Ori is origin of replication. GOI is the gene of interest, which in all plasmids is expressed from a bicistronic mRNA with the appropriate repressor (TetR or LacI). Plasmids pZH713 contains a weaker ribosome binding site than other plasmids. Plasmid pJS101 confers spectinomycin resistance and other plasmids confer ampicillin resistance. </caption>
<thead>
<tr class="header">
<th>Plasmid</th>
<th>Ori</th>
<th>GOI</th>
<th>Promoter</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pZH501</td>
<td>p15a</td>
<td>CI-SNAP-tag</td>
<td>P<sub>LtetO-1</sub></td>
<td><span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span></td>
</tr>
<tr class="even">
<td>pZH509</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td><span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span></td>
</tr>
<tr class="odd">
<td>pJS101</td>
<td>pSC101</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>This work</td>
</tr>
<tr class="even">
<td>pJS102</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LlacO-1</sub></td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pZH713</td>
<td>p15a</td>
<td>PP7cp-SYFP2</td>
<td>P<sub>LlacO-1</sub></td>
<td>This work</td>
</tr>
<tr class="even">
<td>pDG101</td>
<td>p15a</td>
<td>mScarlet-I</td>
<td>P<sub>LlacO-1</sub></td>
<td>This work</td>
</tr>
</tbody>
</table>
<h3 id="characterization-of-gfp-expression-by-flow-cytometry">Characterization of GFP expression by flow cytometry</h3>
<p>All flow cytometry experiments were repeated 3 times on different days and used plasmids transformed by electroporation into <em>E. coli</em> MG1655. Cultures were grown overnight at 30 °C from LB-agar plates supplemented with carbenicillin or spectinomycin (both at 50 <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Cmu%20g%20%2FmL" alt="\mu g /mL" title="\mu g /mL" />) in 1 mL EZ Rich Defined Medium (M2105, Teknova, Hollister) supplemented with the same antibiotics. Cells were reinoculated 1:400 in 1 mL of the same media supplemented with Isopropyl β-D-1-thiogalactopyranoside (IPTG, at concentrations of 0, 2, 4.5, 10, 22.4, 50, 111.8, 250, 559 and 1250 µM) or anhydrotetrycline (ATc, at concentrations of 1, 5, 25 and 125 nM) as indicated and grown at 30 °C for 4—4.5 hours until reaching an optical density at 600 nm of 0.2—0.3. ATc (Alfa Aesar #J66688) is an analog of tetracycline that binds the tetracycline repressor very strongly, allowing it to be used as an inducer in strains without tetracycline resistance <span class="citation" data-cites="8PQGpPEb">[<a href="#ref-8PQGpPEb">34</a>]</span>. Next, 10 µL of cells were added to 1 mL of PBS at pH 7.4 and examined by flow cytometry.</p>
<p>Flow cytometry data was collected on an S3e cell sorter (Bio-Rad, Hercules) using a target flow rate of 2,000 counts per second and collecting 30,000 counts for each sample. A 488-nm laser line was used for excitation at its maximum power setting with amplification settings of 450 (forward scattering, FSC), 350 (side scattering, SSC) and 900 (FL1, 525/30 nm). The cell sorter is calibrated daily for a linear response to sample fluorescence intensity. Acquisition was triggered by forward scattering with a threshold of 3. Data was exported as an FCS file and imported into a custom Python script using FlowCal <span class="citation" data-cites="J6PWaBUu">[<a href="#ref-J6PWaBUu">35</a>]</span>. Following previous methods <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>, one third of samples were selected based on proximity to the peak of FSC-AREA and SSC-HEIGHT in a 2D histogram using the density2d method in FlowCal. The FL1-AREA measurements were used to estimate the mean and variance of GFP distributions for all samples. This was done by estimating the probability density functions in bins distributed equally in logarithmic space and fitting by least squares minimization to a gamma function. We found that this method reduced the influence of low-FL1-AREA events that escaped other gating steps, and which had frequencies that varied for different samples and days (<strong>Figure <a href="#fig:gammaFits">S1</a></strong>). In all plots, the mean fluorescence of a strain harboring a similar plasmid, pZH501, that does not encode a fluorescent protein, was subtracted <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>.</p>
<p>Noise was calculated as the coefficient of variation squared (<img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?CV%5E2" alt="CV^2" title="CV^2" />) from the mean, <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Cmu" alt="\mu" title="\mu" />, and variance, <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Csigma%20%5E2" alt="\sigma ^2" title="\sigma ^2" />, as <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?CV%5E2%20%3D%20%5Csigma%20%5E2%20%2F%20%5Cmu%20%5E2" alt="CV^2 = \sigma ^2 / \mu ^2" title="CV^2 = \sigma ^2 / \mu ^2" />. We chose <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?CV%5E2" alt="CV^2" title="CV^2" /> to facilitate comparison with earlier work <span class="citation" data-cites="12XWQPtGk TlAItren">[<a href="#ref-TlAItren">1</a>,<a href="#ref-12XWQPtGk">36</a>]</span>. We note that gating by FSC-AREA and SSC-HEIGHT to some extent selects for cells near the median cell size, so, ignoring other sources of experimental error, we expect our noise measurements to fall somewhere between the noise in the number of proteins per cell and the noise in the protein concentration. In earlier work identifying the “extrinsic noise limit”, noise was estimated from integrated fluorescence intensities normalized by cell size (proportional to protein concentration) in microscope images <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. The script for data analysis as well as all raw FCS data is available in <strong>File S2</strong> and utilized modules from SciPy, NumPy, Matplotlib and Pandas.</p>
<h3 id="microscopy">Microscopy</h3>
<p>All imaging was done on a Leica DMI6000 inverted microscope using illumination from a Leica EL6000 source (at various intensities ensuring minimal photobleaching during acquisition), fluorescence filter cubes (Leica GFP ET, a custom filter set with Semrock filters FF01-561/13, FF02-616/73 and DI02-R561, or the Semrock LF514-B filter set), a 100x/1.46 a-plan apochromat oil immersion objective, Leica Type F immersion oil, and an Evolve 512 EM-CCD camera (Photometrics) using 16-bit EM gain amplification. Images were prepared using Fiji <span class="citation" data-cites="1Xy8hvPg">[<a href="#ref-1Xy8hvPg">37</a>]</span>, with linear scaling and maintaining minimum and maximum intensity values for all comparable images.</p>
<p>For mRNA imaging, overnight cultures were diluted 1:100 and grown at 30 °C in M9 media supplemented with 1X MEM Amino Acids (M9A) or M9A additionally supplemented with 1% SOB media for 2—4 hours. Supplementation with SOB is used to provide quasi-rich growth conditions with very low fluorescence background and autofluorescence without the expense of commercial rich minimal media. We previously used this growth condition to characterize gene expression noise for different systems <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. Agarose gel pads (3% BP165-25, Fisher Bio-Reagents) were prepared with M9A with and without supplementation with 1% SOB, and the microscope sample chamber was maintained at 30 °C.</p>
<p>To quantify independent induction from 2 plasmids, cells were grown in M9A at 37 °C. Overnight cultures were diluted 1:100 in M9A supplemented with ATc and IPTG and grown for 3.5 hours. At this point, chloramphenicol was added to a final concentration of 100 µg/mL and cultures were incubated for an additional 1 hour at 37 °C to allow most GFPmut2 and mScarlet-I molecules to mature (maturation times of 5.6 and 25.7 minutes, respectively <span class="citation" data-cites="1BqRRwE9k">[<a href="#ref-1BqRRwE9k">38</a>]</span>). Cells were spotted onto agarose gel pads prepared with PBS and imaged at room temperature. For each of 3 replicates and 4 induction conditions, 10 images each were acquired in brightfield, GFPmut2 and mScarlet-I channels. For analysis, all cells in these images were manually segmented using the selection brush tool in Fiji with a width of 10 pixels (163—212 cells per sample). This selection was used to extract the mean green and red intensities (proportional to the concentration of GFPmut2 and mScarlet-I molecules in the cell, respectively). For each image, the mean background intensity was also measured from a large region containing no cells, which was subtracted from each single-cell data point. Mean, variance and <img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?CV%5E2" alt="CV^2" title="CV^2" /> were estimated from this data for each sample following the same fitting protocol used for flow cytometry data. The raw data, single-cell intensities, Fiji macro and Python scripts required to reproduce this analysis are available in <strong>File S2</strong>. Example images were acquired similarly, with growth instead in EZ-Rich media at 30 °C, supplemented with ATc and IPTG.</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JS, SL, DG and ZH designed experiments and performed experiments. JS, SL, DG and ZH performed molecular cloning. ZH, SL and JS analyzed data and wrote the paper. JS, SL, DG and ZH edited and approved the manuscript. ZH supervised the project.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>This work was financially supported by: Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) funded by FEDER funds through COMPETE2020—Programa Operacional Competitividade e Internacionalização (POCI), by national funds through FCT—Fundação para a Ciência e a Tecnologia, and through a joint research agreement with the Okinawa Institute of Science and Technology (OIST). This manuscript was composed and edited using Manubot <span class="citation" data-cites="1GGGHdsew">[<a href="#ref-1GGGHdsew">39</a>]</span>.</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-TlAItren">
<p>1. <strong>A plasmid-based Escherichia coli gene expression system with cell-to-cell variation below the extrinsic noise limit.</strong><br />
Zach Hensel<br />
<em>PloS one</em> (2017-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084263">https://www.ncbi.nlm.nih.gov/pubmed/29084263</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0187259">10.1371/journal.pone.0187259</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084263">29084263</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5662224">PMC5662224</a></p>
</div>
<div id="ref-oD6kpSs3">
<p>2. <strong>Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells</strong><br />
Y. Taniguchi, P. J. Choi, G.-W. Li, H. Chen, M. Babu, J. Hearn, A. Emili, X. S. Xie<br />
<em>Science</em> (2010-07-29) <a href="https://doi.org/c3tkbc">https://doi.org/c3tkbc</a><br />
DOI: <a href="https://doi.org/10.1126/science.1188308">10.1126/science.1188308</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20671182">20671182</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922915">PMC2922915</a></p>
</div>
<div id="ref-GcRoYBF2">
<p>3. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis.</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular systems biology</em> (2006-08-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16883354">https://www.ncbi.nlm.nih.gov/pubmed/16883354</a><br />
DOI: <a href="https://doi.org/10.1038/msb4100081">10.1038/msb4100081</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16883354">16883354</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1681513">PMC1681513</a></p>
</div>
<div id="ref-82diXnad">
<p>4. <strong>An improved MS2 system for accurate reporting of the mRNA life cycle.</strong><br />
Evelina Tutucci, Maria Vera, Jeetayu Biswas, Jennifer Garcia, Roy Parker, Robert H Singer<br />
<em>Nature methods</em> (2017-11-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29131164">https://www.ncbi.nlm.nih.gov/pubmed/29131164</a><br />
DOI: <a href="https://doi.org/10.1038/nmeth.4502">10.1038/nmeth.4502</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29131164">29131164</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5843578">PMC5843578</a></p>
</div>
<div id="ref-5CRbRLYC">
<p>5. <strong>A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.</strong><br />
G Churchward, D Belin, Y Nagamine<br />
<em>Gene</em> (1984-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6098521">https://www.ncbi.nlm.nih.gov/pubmed/6098521</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6098521">6098521</a></p>
</div>
<div id="ref-15TQ1MvYV">
<p>6. <strong>BglBrick vectors and datasheets: A synthetic biology platform for gene expression.</strong><br />
Taek Soon Lee, Rachel A Krupa, Fuzhong Zhang, Meghdad Hajimorad, William J Holtz, Nilu Prasad, Sung Kuk Lee, Jay D Keasling<br />
<em>Journal of biological engineering</em> (2011-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21933410">https://www.ncbi.nlm.nih.gov/pubmed/21933410</a><br />
DOI: <a href="https://doi.org/10.1186/1754-1611-5-12">10.1186/1754-1611-5-12</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21933410">21933410</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189095">PMC3189095</a></p>
</div>
<div id="ref-NCWOk1rp">
<p>7. <strong>Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</strong><br />
R Lutz, H Bujard<br />
<em>Nucleic acids research</em> (1997-03-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630">https://www.ncbi.nlm.nih.gov/pubmed/9092630</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630">9092630</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC146584">PMC146584</a></p>
</div>
<div id="ref-Qz6zNiXK">
<p>8. <strong>Engineering stability in gene networks by autoregulation.</strong><br />
A Becskei, L Serrano<br />
<em>Nature</em> (2000-06-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10850721">https://www.ncbi.nlm.nih.gov/pubmed/10850721</a><br />
DOI: <a href="https://doi.org/10.1038/35014651">10.1038/35014651</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10850721">10850721</a></p>
</div>
<div id="ref-S7KRThVS">
<p>9. <strong>Summing up the noise in gene networks.</strong><br />
Johan Paulsson<br />
<em>Nature</em> (2004-01-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14749823">https://www.ncbi.nlm.nih.gov/pubmed/14749823</a><br />
DOI: <a href="https://doi.org/10.1038/nature02257">10.1038/nature02257</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14749823">14749823</a></p>
</div>
<div id="ref-IbIaRemg">
<p>10. <strong>Two mutations in the tetracycline repressor change the inducer anhydrotetracycline to a corepressor</strong><br />
A. Kamionka<br />
<em>Nucleic Acids Research</em> (2004-01-21) <a href="https://doi.org/fsvj4n">https://doi.org/fsvj4n</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkh200">10.1093/nar/gkh200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14764926">14764926</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC373327">PMC373327</a></p>
</div>
<div id="ref-UMSODjob">
<p>11. <strong>The three operators of the lac operon cooperate in repression.</strong><br />
S Oehler, ER Eismann, H Krämer, B Müller-Hill<br />
<em>The EMBO journal</em> (1990-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2182324">https://www.ncbi.nlm.nih.gov/pubmed/2182324</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2182324">2182324</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC551766">PMC551766</a></p>
</div>
<div id="ref-Ls5d7x4o">
<p>12. <strong>Tet repressor induction without Mg2+.</strong><br />
O Scholz, P Schubert, M Kintrup, W Hillen<br />
<em>Biochemistry</em> (2000-09-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10978179">https://www.ncbi.nlm.nih.gov/pubmed/10978179</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10978179">10978179</a></p>
</div>
<div id="ref-WxFVKQqb">
<p>13. <strong>Induction of the lac promoter in the absence of DNA loops and the stoichiometry of induction.</strong><br />
Stefan Oehler, Siegfried Alberti, Benno Müller-Hill<br />
<em>Nucleic acids research</em> (2006-01-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16432263">https://www.ncbi.nlm.nih.gov/pubmed/16432263</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkj453">10.1093/nar/gkj453</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16432263">16432263</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1345695">PMC1345695</a></p>
</div>
<div id="ref-eUtnfAD6">
<p>14. <strong>ENZYME INDUCTION AS AN ALL-OR-NONE PHENOMENON.</strong><br />
A Novick, M Weiner<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (1957-07-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16590055">https://www.ncbi.nlm.nih.gov/pubmed/16590055</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16590055">16590055</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC528498">PMC528498</a></p>
</div>
<div id="ref-16n3gU60l">
<p>15. <strong>A genome-wide analysis of promoter-mediated phenotypic noise in Escherichia coli.</strong><br />
Olin K Silander, Nela Nikolic, Alon Zaslaver, Anat Bren, Ilya Kikoin, Uri Alon, Martin Ackermann<br />
<em>PLoS genetics</em> (2012-01-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22275871">https://www.ncbi.nlm.nih.gov/pubmed/22275871</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pgen.1002443">10.1371/journal.pgen.1002443</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22275871">22275871</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3261926">PMC3261926</a></p>
</div>
<div id="ref-TvXapFFQ">
<p>16. <strong>Real-time kinetics of gene activity in individual bacteria.</strong><br />
Ido Golding, Johan Paulsson, Scott M Zawilski, Edward C Cox<br />
<em>Cell</em> (2005-12-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16360033">https://www.ncbi.nlm.nih.gov/pubmed/16360033</a><br />
DOI: <a href="https://doi.org/10.1016/j.cell.2005.09.031">10.1016/j.cell.2005.09.031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16360033">16360033</a></p>
</div>
<div id="ref-1GFLV3s4q">
<p>17. <strong>Real-time observation of transcription initiation and elongation on an endogenous yeast gene.</strong><br />
Daniel R Larson, Daniel Zenklusen, Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Science (New York, N.Y.)</em> (2011-04-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21512033">https://www.ncbi.nlm.nih.gov/pubmed/21512033</a><br />
DOI: <a href="https://doi.org/10.1126/science.1202142">10.1126/science.1202142</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21512033">21512033</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152976">PMC3152976</a></p>
</div>
<div id="ref-uQeM79z">
<p>18. <strong>Spatial organization of the flow of genetic information in bacteria.</strong><br />
Paula Montero Llopis, Audrey F Jackson, Oleksii Sliusarenko, Ivan Surovtsev, Jennifer Heinritz, Thierry Emonet, Christine Jacobs-Wagner<br />
<em>Nature</em> (2010-06-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20562858">https://www.ncbi.nlm.nih.gov/pubmed/20562858</a><br />
DOI: <a href="https://doi.org/10.1038/nature09152">10.1038/nature09152</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20562858">20562858</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896451">PMC2896451</a></p>
</div>
<div id="ref-zj8Ydkzo">
<p>19. <strong>Characterization of a spectrally diverse set of fluorescent proteins as FRET acceptors for mTurquoise2.</strong><br />
Marieke Mastop, Daphne S Bindels, Nathan C Shaner, Marten Postma, Theodorus WJ Gadella, Joachim Goedhart<br />
<em>Scientific reports</em> (2017-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28931898">https://www.ncbi.nlm.nih.gov/pubmed/28931898</a><br />
DOI: <a href="https://doi.org/10.1038/s41598-017-12212-x">10.1038/s41598-017-12212-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28931898">28931898</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5607329">PMC5607329</a></p>
</div>
<div id="ref-YoMCfbca">
<p>20. <strong>Isolation of the lac repressor.</strong><br />
W Gilbert, B Müller-Hill<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (1966-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16591435">https://www.ncbi.nlm.nih.gov/pubmed/16591435</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16591435">16591435</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC220206">PMC220206</a></p>
</div>
<div id="ref-N7fJlkrJ">
<p>21. <strong>Broad host range plasmids.</strong><br />
Aayushi Jain, Preeti Srivastava<br />
<em>FEMS microbiology letters</em> (2013-09-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23980652">https://www.ncbi.nlm.nih.gov/pubmed/23980652</a><br />
DOI: <a href="https://doi.org/10.1111/1574-6968.12241">10.1111/1574-6968.12241</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23980652">23980652</a></p>
</div>
<div id="ref-pzTydEMA">
<p>22. <strong>Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.</strong><br />
Amin Espah Borujeni, Anirudh S Channarasappa, Howard M Salis<br />
<em>Nucleic acids research</em> (2013-11-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24234441">https://www.ncbi.nlm.nih.gov/pubmed/24234441</a><br />
DOI: <a href="https://doi.org/10.1093/nar/gkt1139">10.1093/nar/gkt1139</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24234441">24234441</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3936740">PMC3936740</a></p>
</div>
<div id="ref-xjKSoV9v">
<p>23. <strong>Intrinsic termination of T7 RNA polymerase mediated by either RNA or DNA.</strong><br />
L Hartvig, J Christiansen<br />
<em>The EMBO journal</em> (1996-09-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8887568">https://www.ncbi.nlm.nih.gov/pubmed/8887568</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8887568">8887568</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC452209">PMC452209</a></p>
</div>
<div id="ref-12aHdKu6v">
<p>24. <strong>Production of Protein-Complex Components Is Stoichiometric and Lacks General Feedback Regulation in Eukaryotes.</strong><br />
James C Taggart, Gene-Wei Li<br />
<em>Cell systems</em> (2018-12-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30553725">https://www.ncbi.nlm.nih.gov/pubmed/30553725</a><br />
DOI: <a href="https://doi.org/10.1016/j.cels.2018.11.003">10.1016/j.cels.2018.11.003</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30553725">30553725</a></p>
</div>
<div id="ref-5dC8qsZ9">
<p>25. <strong>Comparative Single-Cell Analysis of Different E. coli Expression Systems during Microfluidic Cultivation.</strong><br />
Dennis Binder, Christopher Probst, Alexander Grünberger, Fabienne Hilgers, Anita Loeschcke, Karl-Erich Jaeger, Dietrich Kohlheyer, Thomas Drepper<br />
<em>PloS one</em> (2016-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27525986">https://www.ncbi.nlm.nih.gov/pubmed/27525986</a><br />
DOI: <a href="https://doi.org/10.1371/journal.pone.0160711">10.1371/journal.pone.0160711</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27525986">27525986</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4985164">PMC4985164</a></p>
</div>
<div id="ref-dc6GAnhL">
<p>26. <strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong><br />
Daniel G Gibson, Lei Young, Ray-Yuan Chuang, J Craig Venter, Clyde A Hutchison, Hamilton O Smith<br />
<em>Nature Methods</em> (2009-04-12) <a href="https://doi.org/ckn92w">https://doi.org/ckn92w</a><br />
DOI: <a href="https://doi.org/10.1038/nmeth.1318">10.1038/nmeth.1318</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19363495">19363495</a></p>
</div>
<div id="ref-13TJF2gcl">
<p>27. <strong>FACS-optimized mutants of the green fluorescent protein (GFP).</strong><br />
BP Cormack, RH Valdivia, S Falkow<br />
<em>Gene</em> (1996) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8707053">https://www.ncbi.nlm.nih.gov/pubmed/8707053</a><br />
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8707053">8707053</a></p>
</div>
<div id="ref-Q1a7tU8">
<p>28. <strong>Nucleotide sequence of the repressor gene of the TN10 tetracycline resistance determinant</strong><br />
Kathleen Postle, Toai T. Nguyen, Kevin P. Bertrand<br />
<em>Nucleic Acids Research</em> (1984) <a href="https://doi.org/cvpfkh">https://doi.org/cvpfkh</a><br />
DOI: <a href="https://doi.org/10.1093/nar/12.12.4849">10.1093/nar/12.12.4849</a></p>
</div>
<div id="ref-A1kNMUKB">
<p>29. <strong>Cell-cycle-synchronized, oscillatory expression of a negatively autoregulated gene in E. coli</strong><br />
Zach Hensel, Tatiana T. Marquez-Lago<br />
<em>arXiv</em> (2015-06-29) <a href="https://arxiv.org/abs/1506.08596v1">https://arxiv.org/abs/1506.08596v1</a></p>
</div>
<div id="ref-RBKXt78E">
<p>30. <strong>One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.</strong><br />
KA Datsenko, BL Wanner<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (2000-06-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079">https://www.ncbi.nlm.nih.gov/pubmed/10829079</a><br />
DOI: <a href="https://doi.org/10.1073/pnas.120163297">10.1073/pnas.120163297</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079">10829079</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC18686">PMC18686</a></p>
</div>
<div id="ref-XuRR8Zmt">
<p>31. <strong>Cyan and yellow super fluorescent proteins with improved brightness, protein folding, and FRET Förster radius.</strong><br />
Gert-Jan Kremers, Joachim Goedhart, Erik B van Munster, Theodorus WJ Gadella<br />
<em>Biochemistry</em> (2006-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16716067">https://www.ncbi.nlm.nih.gov/pubmed/16716067</a><br />
DOI: <a href="https://doi.org/10.1021/bi0516273">10.1021/bi0516273</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16716067">16716067</a></p>
</div>
<div id="ref-EObeLKJ1">
<p>32. <strong>Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells.</strong><br />
Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Biophysical journal</em> (2012-06-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22735544">https://www.ncbi.nlm.nih.gov/pubmed/22735544</a><br />
DOI: <a href="https://doi.org/10.1016/j.bpj.2012.05.017">10.1016/j.bpj.2012.05.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22735544">22735544</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3379624">PMC3379624</a></p>
</div>
<div id="ref-q8xLcfYV">
<p>33. <strong>Zach Hensel Lab Plasmids</strong><br />
Zach Hensel, João Silva<br />
<em>AddGene</em> (2019) <a href="https://www.addgene.org/Zach_Hensel/">https://www.addgene.org/Zach_Hensel/</a></p>
</div>
<div id="ref-8PQGpPEb">
<p>34. <strong>Tetracycline analogs affecting binding to Tn10-Encoded Tet repressor trigger the same mechanism of induction.</strong><br />
T Lederer, M Kintrup, M Takahashi, PE Sum, GA Ellestad, W Hillen<br />
<em>Biochemistry</em> (1996-06-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8652521">https://www.ncbi.nlm.nih.gov/pubmed/8652521</a><br />
DOI: <a href="https://doi.org/10.1021/bi952683e">10.1021/bi952683e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8652521">8652521</a></p>
</div>
<div id="ref-J6PWaBUu">
<p>35. <strong>FlowCal: A User-Friendly, Open Source Software Tool for Automatically Converting Flow Cytometry Data from Arbitrary to Calibrated Units.</strong><br />
Sebastian M Castillo-Hair, John T Sexton, Brian P Landry, Evan J Olson, Oleg A Igoshin, Jeffrey J Tabor<br />
<em>ACS synthetic biology</em> (2016-05-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27110723">https://www.ncbi.nlm.nih.gov/pubmed/27110723</a><br />
DOI: <a href="https://doi.org/10.1021/acssynbio.5b00284">10.1021/acssynbio.5b00284</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27110723">27110723</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5556937">PMC5556937</a></p>
</div>
<div id="ref-12XWQPtGk">
<p>36. <strong>Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells.</strong><br />
Yuichi Taniguchi, Paul J Choi, Gene-Wei Li, Huiyi Chen, Mohan Babu, Jeremy Hearn, Andrew Emili, X Sunney Xie<br />
<em>Science (New York, N.Y.)</em> (2010-07-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20671182">https://www.ncbi.nlm.nih.gov/pubmed/20671182</a><br />
DOI: <a href="https://doi.org/10.1126/science.1188308">10.1126/science.1188308</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20671182">20671182</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2922915">PMC2922915</a></p>
</div>
<div id="ref-1Xy8hvPg">
<p>37. <strong>Fiji: an open-source platform for biological-image analysis.</strong><br />
Johannes Schindelin, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, Curtis Rueden, Stephan Saalfeld, Benjamin Schmid, … Albert Cardona<br />
<em>Nature methods</em> (2012-06-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772">https://www.ncbi.nlm.nih.gov/pubmed/22743772</a><br />
DOI: <a href="https://doi.org/10.1038/nmeth.2019">10.1038/nmeth.2019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772">22743772</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3855844">PMC3855844</a></p>
</div>
<div id="ref-1BqRRwE9k">
<p>38. <strong>Systematic characterization of maturation time of fluorescent proteins in living cells.</strong><br />
Enrique Balleza, J Mark Kim, Philippe Cluzel<br />
<em>Nature methods</em> (2017-11-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29320486">https://www.ncbi.nlm.nih.gov/pubmed/29320486</a><br />
DOI: <a href="https://doi.org/10.1038/nmeth.4509">10.1038/nmeth.4509</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29320486">29320486</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5765880">PMC5765880</a></p>
</div>
<div id="ref-1GGGHdsew">
<p>39. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>Manubot Preprint</em> (2018) <a href="https://greenelab.github.io/meta-review/">https://greenelab.github.io/meta-review/</a></p>
</div>
<div id="ref-PgIr1Yp">
<p>40. <strong>Supplementary File S2: Plasmids for independently tunable, low-noise gene expression</strong><br />
João P. N. Silva, Soraia Vidigal Lopes, Diogo J. Grilo, Zach Hensel<br />
<em>Zenodo</em> (2019-01-09) <a href="https://doi.org/gf2bw6">https://doi.org/gf2bw6</a><br />
DOI: <a href="https://doi.org/10.5281/zenodo.2671633">10.5281/zenodo.2671633</a></p>
</div>
</div>
<h2 id="supplementary-material" class="page_break_before">Supplementary Material</h2>
<figure>
<img src="images/FigureS1_gammaFits.png" alt="Figure S1: Reproducibility of low-noise expression in 3 independent experiments. Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples. All plots are normalized by the maximum value of the fit gamma distribution." id="fig:gammaFits" data-tag="S1" style="width:89mm" /><figcaption><span>Figure S1:</span> <strong>Reproducibility of low-noise expression in 3 independent experiments.</strong> Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples. All plots are normalized by the maximum value of the fit gamma distribution.</figcaption>
</figure>
<p><strong>Supplementary File S2</strong>: Raw flow cytometry data, Python scripts required to reproduce Figures <a href="#fig:pZH509-pJS101">1</a>, <a href="#fig:pJS102-mean-noise">2</a>, and <a href="#fig:pJS101-pDG101">4</a>, DNA sequences, and explanatory text files are available as a compressed archive at Zenodo <span class="citation" data-cites="PgIr1Yp">[<a href="#ref-PgIr1Yp">40</a>]</span>.</p>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Open Sans", "Helvetica", sans-serif;

        /* prevent text inflation on some mobile browsers */
        -webkit-text-size-adjust: none !important;
        -moz-text-size-adjust: none !important;
        -o-text-size-adjust: none !important;
        text-size-adjust: none !important;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            max-width: 8.5in;
            font-size: 12pt;
            line-height: 2.0;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            word-break: all;
            word-break: break-word;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin-top: 40px;
        margin-bottom: 30px;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 0;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        /* put line break break anywhere in text */
        word-break: break-all;
        /* put line break between words. if not supported by browser, will fall back to break-all */
        word-break: break-word;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #f5f5f5;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.75in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* <h2> heading */
        h2 {
            margin-top: 0;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number */
        margin-right: 40px !important;
    }

    /* -------------------------------------------------- */
    /* table scroll plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* table wrapper */
        .table_wrapper {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow: auto;
            width: 100%;
            margin: 20px 0;
        }

        /* table within table wrapper */
        .table_wrapper table,
        .table_wrapper table * {
            /* don't break table words */
            word-break: normal !important;
        }

        .table_wrapper > table {
            /* move margins from table to table_wrapper to allow margin collapsing */
            margin: 0;
        }
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin: 10px 0;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            word-break: all;
            word-break: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
        }

        /* tooltip copy of figure captions */
        #tooltip_content > figure > figcaption {
            margin: 0;
            margin-top: 20px;
            padding: 0;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel when open */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            background: #ffffff;
            border-right: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_panel > h3 {
            box-sizing: border-box;
            height: 60px;
            margin: 0;
            padding: 20px;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when hovered */
        #toc_panel > h3:hover {
            cursor: pointer;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
            vertical-align: top;
        }

        /* toc header text */
        #toc_header_text {
            vertical-align: top;
            font-weight: 600;
            position: relative;
            top: -4px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] #toc_header_text,
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
</body>
</html>
