---
author-meta:
- "Jo\xE3o Silva"
- Soraia Lopes
- Diogo Grilo
- Zach Hensel
date-meta: '2019-01-03'
keywords:
- gene expression
- synthetic biology
- transcription
lang: en-US
title: Plasmids for independently tunable, low-noise gene expression
...






<small><em>
This manuscript
([permalink](https://zach-hensel.github.io/low-noise-manuscript/v/49084d1790d4e8a2abeeeb8070445e2dbcdc9861/))
was automatically generated
from [zach-hensel/low-noise-manuscript@49084d1](https://github.com/zach-hensel/low-noise-manuscript/tree/49084d1790d4e8a2abeeeb8070445e2dbcdc9861)
on January 3, 2019.
</em></small>

## Authors



+ **João Silva**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)<br>
  <small>
     Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal
  </small>

+ **Soraia Lopes**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)<br>
  <small>
     Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal
  </small>

+ **Diogo Grilo**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [XXXX-XXXX-XXXX-XXXX](https://orcid.org/XXXX-XXXX-XXXX-XXXX)<br>
  <small>
     Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal
  </small>

+ **Zach Hensel**<br>
    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
    [0000-0002-4348-6229](https://orcid.org/0000-0002-4348-6229)
    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
    [zach-hensel](https://github.com/zach-hensel)
    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
    [alchemytoday](https://twitter.com/alchemytoday)<br>
  <small>
     Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal
     · Funded by Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular)
    <br>Correspondence: [zach.hensel@itqb.unl.pt](mailto:zach.hensel@itqb.unl.pt)
  </small>



## Abstract {.page_break_before}

Some microbiology experiments and biotechnology applications can be improved if it is possible to tune the expression of two different genes at the same time with cell-to-cell variation at or below the level of genes constitutively expressed from the chromosome (the “extrinsic noise limit”). This was recently achieved for a single gene by exploiting negative autoregulation by the tetracycline repressor (TetR), and bicistronic gene expression to reduce gene expression noise. We report new plasmids that use the same principles to achieve simultaneous, low-noise expression for two genes. The TetR system was moved to a compatible plasmid backbone, and a system based on the lactose repressor (LacI) was found to also exhibit gene expression noise below the extrinsic noise limit. We characterized gene expression mean and noise across the range of induction levels for these strains, applied the LacI system to tune expression for single-molecule mRNA detection in two different growth conditions, and showed that two plasmids can be co-transformed to independently tune expression of two different genes.

## Introduction {.page_break_before}

We recently reported the development of a plasmid-based gene-expression system in which a gene of interest is expressed bicistronically with the tetracycline repressor (TetR) [@RB57dahw]. Using this gene expression system, cell-to-cell is below the “extrinsic noise limit” observed for genes expressed from the chromosome [@oD6kpSs3]. When TetR and GFP are expressed bicistronically, GFP induction and gene expression noise is similar to that observed for a TetR-GFP fusion protein with autoregulation [@x1Hd2npA]. Compared to induction of gene expression under the control of a constitutively expressed transcriptional repressor, the inducer dose-response is relatively linearized, and gene expression noise is much lower at intermediate induction levels.

Why this would it be useful to do this for two different genes at the same time?
Example for stoichiometric production of a protein complex
Expressing genes at their native levels … e.g. for generating chromosome deletions

Figure 1: Regulatory constructs used in this study. (a) pZH509, (b) pJS102, (c) pJS103, (d) pJS23103

## Methods 

### Strain construction

All plasmids were constructed using isothermal assembly [@dc6GAnhL] of fragments generated by PCR or double stranded DNA synthesis (IDT) and transformed Top10 E. coli cells (IBA #5-1600-020). Transformants were screened by colony PCR and verified by sequencing. Sequence maps are included as supplementary files and plasmids will be made available at AddGene after deposit and screening [@q8xLcfYV]. Purified plasmids were transformed into E. coli strain MG1655 by growing 2 mL of culture in SOB media at 37 C to OD600=0.4, washing twice with 1 mL ice-cold water, resuspending in 40 µL water, electroporation of 1–10 ng plasmid with the EC1 setting of a Micropulser (BioRad), and recovering for 1 hour at 37 C in SOC media.

To generate pJS101 with a compatible backbone, plasmid pZH509 [@RB57dahw] was used as a template to amplify the bicistronic regulatory construct including the P~LtetO-1~ promoter [@NCWOk1rp], GFPmut2 [@13TJF2gcl], tn10 TetR [@Q1a7tU8] and rrnB T1 transcription terminator [@xjKSoV9v]. This was assembled by isothermal assembly with the backbone from pGB2 [@5CRbRLYC] with the pSC101 origin of replication and spectinomycin resistance to generate plasmid pJS101. Plasmids with the pSC101 and p15a origins of replication have been co-transformed in previous synthetic biology experiments [@fyyCRa8e].

Plasmids pJS102 and pJS103 were generated by 3-fragment isothermal assembly. Plasmid pZH509 was used as a template both for the vector backbone and for GFPmut2, with non-homologous extensions added to PCR primers to generate the P~LlacO~ and P~LlacOsym~ promoters. Promoter P~LlacOsym~ lacks the central G–C base pair in P~LlacO~ LacI binding sites, which are symmetrical [@swicCLW1] except that the second *lacOsym* sequence has one base changed to maintain the P~LlacO~ promoter –10 sequence. LacI was amplified from *E. coli* MG1655 by colony PCR. Plasmid pJS23103 was made by isothermal assembly of an inverse PCR product of pZH509, with non-homologous primer extensions adding the weak, constitutive BBa_J23103 promoter between GFPmut2 and TetR [@ZZUG1lTS].

Plasmids were co-transformed into MG1655 by electroporation following the above protocol, except with 1 µL each undiluted plasmid (~20–40 ng) and selecting on plates with both spectinomycin and carbenicillin.

| Plasmid | Ori | Promoter | GOI | Repressor Expression | Reference |
|----------|----------|----------|----------|----------|----------|
| pZH501 | p15a | CI-SNAP-tag | ~LtetO-1~ | bicistronic | [@RB57dahw] |
| pZH509 | p15a | GFPmut2 |  P~LtetO-1~ | bicistronic | [@RB57dahw] |
| pZH520 | p15a | GFPmut2 | P~LtetO-1~ | constitutive | [@RB57dahw] |
| pJS101 | pSC101 | GFPmut2 | P~LtetO-1~ | bicistronic | This work |
| pJS102 | p15a | GFPmut2 | P~LlacO~ | bicistronic | This work |
| pJS103 | p15a | GFPmut2 | P~LlacOsym~ | bicistronic | This work |
| pJS23103 | p15a | GFPmut2 | P~LtetO-1~ | bicistronic/constitutive | This work |
| pZH742 | pSC101 | PP7cp-mNeonGreen | P~LlacO~  | bicistronic | This work |
| pZH740 | p15a | mScarlet-I-24xPP7sl | P~LtetO-1~  | constitutive | This work |

Table: Plasmids used in this study. {#tbl:plasmidList}

### Cell Growth

Cells were grown in overnight cultures at 32 C in M9 minimal media supplemented with 1% SOB media, 50 µg/mL carbenicillin or 50 µg/mL spectinomycin. The next day, cells were diluted 1:100 or more (if multiple experiments were to be done sequentially over several hours) in the same media supplemented with anhydrotetracycline. In this growth condition, doubling time was approximately 30 minutes for MG1655 strains harboring these plasmids.

### Microscopy

Microscopy methods and microscopy data analysis methods

### Flow cytometry

Flow cytometry methods and data analysis methods

## Results 

### Moving bicistronic autoregulatory construct to a compatible plasmid backbone

The first step in creating a low-noise system for tuning expression of two genes is to establish that a previously characterized bicistronic autoregulatory circuit functions well in a compatible plasmid backbone. In this expression system, GFP and TetR are expressed bicistronically from the TetR-repressible promoter P~LtetO-1~ and expression is induced by the addition of ATc [@TlAItren]. This system was shown to have low noise and a linearized dose response compared to a system in which TetR is constitutively expressed. We moved the system from a plasmid with a p15A replicon conferring ampicillin resistance to a lower-copy-number plasmid with a pSC101 replicon conferring spectinomycin resistance [@5CRbRLYC]. The p15A and pSC101 replicons have been used together in multiplasmid systems [@15TQ1MvYV].

[@TlAItren] my plosone paper
[@15TQ1MvYV] keasling biobricks

GFP expression mean and noise were characterized from low to high levels of induction by flow cytometry. Figure {@fig:pZH509-pJS101} shows that pJS101 induces at similar ATc concentrations as pZH509, with the change to the lower-copy pSC101 backbone resulting in a 58% drop in mean expression levels at a wide range of ATc concentrations. For a similar expression system in the absence of autoregulated TetR expression, moving the P~LtetO-1~ promoter from a p15A to a pSC101 backbone resulted in an 87% drop in expression [@NCWOk1rp]. A smaller change is expected in our experiment since negative autoregulation will provide dosage compensation, just autoregulation can reduce noise in plasmid copy number [@GcRoYBF2; @Qz6zNiXK; @S7KRThVS].

[@GcRoYBF2] dublanche copy number variation
[@Qz6zNiXK] becskei serrano autoregulation noise
[@S7KRThVS] paulsson summing up the noise
[@NCWOk1rp] lutz laco1 etc

![**Influence of plasmid backbone on induction of TetR expression system.** Mean single-cell GFP fluorescence for pZH509 (blue) and pJS101 (orange) plasmids as a function of ATc concentration. Error bars are 1 standard error. ](images/Fig1_pZH509_pJS101.png){#fig:pZH509-pJS101}

### Alternative regulatory constructs with LacI replacing TetR

We hypothesized that replacing P~LtetO-1~ with the inducible promoter P~LlacO-1~  with similar characteristics [@NCWOk1rp] and replacing TetR with LacI might result in a similarly useful expression system that could be tuned independently. However, regulatory parameters for TetR and LacI vary wildly, with ...

binding constants:
tetR:
LacI:

half induction:
TetR ATc Tc
LacI IPTG

Figure {@fig:pJS102-mean-noise} shows 

![**Characterizing mean expression levels and noise for different gene expression systems.** (**a**) pJS102 mean induction (**b**) mean vs noise for all strains.](images/Fig2_pJS102_mean_noise.png){#fig:pJS102-mean-noise}

### Using the new induction system for detection of single mRNA in living *E. coli*

blah blah

Figure {@fig:pZH713-optimization} shows 

![**Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.** (**a**) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (**b**) Using the pJS102 expression system, an optimal range of 10--20 µM IPTG is quickly identified for single mRNA detection in minimal media.](images/Fig3_pZH713_optimization.png){#fig:pZH713-optimization}


### Independent, tunable expression of two genes

Aggregation of mRNA-binding proteins from RNA bacteriophages makes it difficult to use fluorescent fusion proteins to detect single mRNAs [@cfHwILHp]. We hypothesized that ...

[@ne2ewtkS] mNeonGreen
[@H1cK0KNl] mScarlet-I
[@oLbIsF1b] The Pf3 coat protein is a small membrane-spanning protein with a cytoplasm-facing C terminus.

Figure {@fig:pJS101-pDG101} shows 

![**Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.** (**a**) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (**b**) Using the pJS102 expression system, an optimal range of 10--20 µM IPTG is quickly identified for single mRNA detection in minimal media.](images/Fig4_pJS101_pDG101.png){#fig:pJS101-pDG101}

## Discussion

Possible to improve pJS23103 with somewhat stronger promoter?

Possible to extend to 3 genes? Possible using additional repressor, pMB1 origin of replication, and different antibiotic selection.

## Author contributions

JS and ZH designed experiments, performed experiments, analyzed data, and wrote the paper. ZH supervised the project.

## Acknowledgments

This work was financially supported by: Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) funded by FEDER funds through COMPETE2020—Programa Operacional Competitividade e Internacionalização (POCI), by national funds through FCT—Fundação para a Ciência e a Tecnologia, and through a joint research agreement with the Okinawa Institute of Science and Technology (OIST).

## References {.page_break_before}

<!-- Explicitly insert bibliography here -->
<div id="refs"></div>
