<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="João P. N. Silva" />
  <meta name="author" content="Soraia Vidigal Lopes" />
  <meta name="author" content="Diogo J. Grilo" />
  <meta name="author" content="Zach Hensel" />
  <meta name="dcterms.date" content="2019-01-07" />
  <meta name="keywords" content="gene expression, synthetic biology, transcription" />
  <title>Plasmids for independently tunable, low-noise gene expression</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css">
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header>
<h1 class="title">Plasmids for independently tunable, low-noise gene expression</h1>
</header>
<p><small><em> This manuscript (<a href="https://zach-hensel.github.io/low-noise-manuscript/v/676d500af117a19ce116d06762324bf7b6021b80/">permalink</a>) was automatically generated from <a href="https://github.com/zach-hensel/low-noise-manuscript/tree/676d500af117a19ce116d06762324bf7b6021b80">zach-hensel/low-noise-manuscript@676d500</a> on January 7, 2019. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>João P. N. Silva</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-4455-4286">0000-0003-4455-4286</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Soraia Vidigal Lopes</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-3705-2619">0000-0002-3705-2619</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Diogo J. Grilo</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0003-1279-3629">0000-0003-1279-3629</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Zach Hensel</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-4348-6229">0000-0002-4348-6229</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/zach-hensel">zach-hensel</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal · Funded by Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) <br>Correspondence: <a href="mailto:zach.hensel@itqb.unl.pt">zach.hensel@itqb.unl.pt</a> </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Some microbiology experiments and biotechnology applications can be improved if it is possible to tune the expression of two different genes at the same time with cell-to-cell variation at or below the level of genes constitutively expressed from the chromosome (the “extrinsic noise limit”). This was recently achieved for a single gene by exploiting negative autoregulation by the tetracycline repressor (TetR) and bicistronic gene expression to reduce gene expression noise. We report new plasmids that use the same principles to achieve simultaneous, low-noise expression for two genes. The TetR system was moved to a compatible plasmid backbone, and a system based on the lactose repressor (LacI) was found to also exhibit gene expression noise below the extrinsic noise limit. We characterize gene expression mean and noise across the range of induction levels for these plasmids, apply the LacI system to tune expression for single-molecule mRNA detection in two different growth conditions, and show that two plasmids can be co-transformed to independently tune expression of two different genes.</p>
<h2 id="introduction" class="page_break_before">Introduction</h2>
<p>We recently reported the development of a plasmid-based gene-expression system in which a gene of interest is expressed bicistronically with the tetracycline repressor (TetR) <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. Using this gene expression system, cell-to-cell variation is below the “extrinsic noise limit” (coefficient of variation squared of protein concentration, <span class="math inline">\(CV^2 \approx 0.1\)</span>) observed for genes expressed from the chromosome <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>. When TetR and GFP are expressed bicistronically, GFP induction and gene expression noise is similar to that observed for a TetR-GFP fusion protein with autoregulation <span class="citation" data-cites="x1Hd2npA">[<a href="#ref-x1Hd2npA">3</a>]</span>. Compared to induction of gene expression under the control of a constitutively expressed transcriptional repressor, the inducer dose-response is relatively linearized, and gene expression noise is much lower at intermediate induction levels <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>.</p>
<p>Our recent experiments in mRNA detection and other single-molecule experiments in living <em>E. coli</em> cells sometimes require the tunable expression of two different genes, both with low noise levels. For example, adopting a recently reported mRNA detection systems based on local enrichment of fluorescent RNA-binding proteins <span class="citation" data-cites="82diXnad">[<a href="#ref-82diXnad">4</a>]</span> for use in <em>E. coli</em> requires lower noise in protein production relative to the same system in <em>S. cerevisiae</em>, because of a smaller cell volume and the inability to sequester unbound protein in the nucleus. At the same time, tunable expression with low noise in the level of the target RNA is desired to make it possible to characterize the accuracy of RNA detection over a range of RNA levels. We hoped that expressing both the target RNA and RNA-binding fluorescent protein on two plasmids that could be tuned independently would simplify and accelerate development of new RNA-detection systems in <em>E. coli</em>. Achieving this was a three-step process: first, characterizing the TetR-based system on a compatible plasmid backbone; second, establishing an orthogonal, low-noise expression system based on the <em>lac</em> repressor (LacI); and third, showing that the two systems can be tuned independently.</p>
<h2 id="methods">Methods</h2>
<h3 id="strain-construction">Strain construction</h3>
<p>All plasmids were constructed using isothermal assembly <span class="citation" data-cites="dc6GAnhL">[<a href="#ref-dc6GAnhL">5</a>]</span> of fragments generated by PCR or double stranded DNA synthesis (IDT, Coralville) and transformed into Top10 E. coli cells (5-1600-020, IBA Life Sciences, Göttingen). Transformants were screened by colony PCR and verified by sequencing. Purified plasmids were transformed into <em>E. coli</em> strain MG1655 by growing 3 mL of culture in SOB media at 30 °C to an optical density at 600 nm (OD600) of 0.4, washing twice with 1 mL ice-cold water, resuspending in 40 µL water, electroporating 1—10 ng plasmid with the EC1 setting of a Micropulser (Bio-Rad Laboratories, Hercules), and recovering for 1 hour at 37 °C in SOC media before plating on selective LB-agar.</p>
<p>To generate pJS101 with a compatible backbone, plasmid pZH509 <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span> was used as a template to amplify the bicistronic regulatory construct including the P<sub>LtetO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>, GFPmut2 <span class="citation" data-cites="13TJF2gcl">[<a href="#ref-13TJF2gcl">7</a>]</span>, tn10 TetR <span class="citation" data-cites="Q1a7tU8">[<a href="#ref-Q1a7tU8">8</a>]</span> and rrnB T1 transcription terminator <span class="citation" data-cites="xjKSoV9v">[<a href="#ref-xjKSoV9v">9</a>]</span>. This was assembled by isothermal assembly with the backbone from pGB2 <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">10</a>]</span> with the pSC101 origin of replication and spectinomycin resistance to generate plasmid pJS101. Plasmid pJS102 was generated by 3-fragment isothermal assembly. Plasmid pZH509 was used as a template both for the vector backbone and for GFPmut2, with non-homologous extensions added to PCR primers to generate the P<sub>LlacO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>. LacI <span class="citation" data-cites="YoMCfbca">[<a href="#ref-YoMCfbca">11</a>]</span> was amplified from <em>E. coli</em> MG1655 by colony PCR.</p>
<p>The test strain for mRNA imaging, ZHX99, was constructed similarly to ZHX222 in recent work <span class="citation" data-cites="A1kNMUKB">[<a href="#ref-A1kNMUKB">12</a>]</span>. In ZHX99, a construct in which a fusion protein of mVenus and Cro is expressed from the bacteriophage λ promoter <em>P<sub>R</sub></em> was integrated into the chromosome to replace the <em>lac</em> operon in MG1655 <span class="citation" data-cites="RBKXt78E">[<a href="#ref-RBKXt78E">13</a>]</span>. ZHX99 differs from ZHX222 in three ways. First, the <em>P<sub>R</sub></em> promoter was weakened by site-directed mutagenesis to produce a strain with lower mRNA levels. Second, a very strong ribosome binding site was added (RBS #136 <span class="citation" data-cites="pzTydEMA">[<a href="#ref-pzTydEMA">14</a>]</span>). Lastly, 24 tandem repeats of the recognition sequence for the PP7 coat protein (PP7cp) were inserted between the open reading frame and transcription terminator (amplified by PCR from pDZ251 <span class="citation" data-cites="1GFLV3s4q">[<a href="#ref-1GFLV3s4q">15</a>]</span>). The pZH713 plasmid for mRNA detection was constructed by replacing GFPmut2 in pJS102 with a fusion protein of SYFP2 (amplified from a plasmid <span class="citation" data-cites="XuRR8Zmt">[<a href="#ref-XuRR8Zmt">16</a>]</span>) and PP7cp (generated by codon optimization and DNA synthesis based on previously reported sequences <span class="citation" data-cites="EObeLKJ1">[<a href="#ref-EObeLKJ1">17</a>]</span>). Additionally, in pZH713 the PP7cp-SYFP2 fusion protein is translated from the weak ribosome binding site from pZH511 <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. We note that mVenus expression in ZHX99 is extremely low (undetectable without strong laser excitation) and does not interfere with mRNA detection by localizing up to 48 SYFP2 molecules in a diffraction-limited spot bound to a single mRNA molecule.</p>
<p>To test independent induction of two genes, GFPmut2 in pJS102 was replaced by mScarlet-I (amplified from a plasmid <span class="citation" data-cites="zj8Ydkzo">[<a href="#ref-zj8Ydkzo">18</a>]</span>) to make pDG101. Plasmids were co-transformed into MG1655 by electroporation following the above protocol, except with 1 µL each undiluted plasmid (20—40 ng) and selecting on LB-agar plates with both spectinomycin and carbenicillin. Sequence maps are included in <strong>File S2</strong> and plasmids useful for constructing additional two-gene expression systems (pJS101 and pJS102, <strong>Table <a href="#tbl:plasmidList">1</a></strong>) are available from AddGene (deposit numbers 118280 and 118281) and have been verified by NGS <span class="citation" data-cites="q8xLcfYV">[<a href="#ref-q8xLcfYV">19</a>]</span>.</p>
<a name="tbl:plasmidList"></a>
<table>
<caption><span>Table 1:</span> Plasmids used in this study. </caption>
<thead>
<tr class="header">
<th>Plasmid</th>
<th>Ori</th>
<th>Promoter</th>
<th>GOI</th>
<th>Repressor Expression</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pZH501</td>
<td>p15a</td>
<td>CI-SNAP-tag</td>
<td><sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span></td>
</tr>
<tr class="even">
<td>pZH509</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span></td>
</tr>
<tr class="odd">
<td>pJS101</td>
<td>pSC101</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pJS102</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pZH713</td>
<td>p15a</td>
<td>PP7cp-SYFP2</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pDG101</td>
<td>p15a</td>
<td>mScarlet-I</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
</tbody>
</table>
<h3 id="characterization-of-gfp-expression-by-flow-cytometry">Characterization of GFP expression by flow cytometry</h3>
<p>All flow cytometry experiment were repeated 3 times on different days and used plasmids transformed by electroporation into <em>E. coli</em> MG1655. Cultures were grown overnight at 30 °C from LB-agar plates supplemented with carbenicillin or spectinomycin (both at 50 <span class="math inline">\(\mu g /mL\)</span>) in 1 mL EZ Rich Defined Medium (M2105, Teknova, Hollister) supplemented with the same antibiotics. Cells were reinoculated 1:400 in 1 mL of the same media supplemented with Isopropyl β-D-1-thiogalactopyranoside (IPTG, at concentrations of 0, 2, 4.5, 10, 22.4, 50, 111.8, 250, 559, and 1250 µM) or anhydrotetrycline (ATc, at concentrations of 1, 5, 25, and 125 nM) as indicated and grown at 30 °C for 4—4.5 hours until reaching an optical density at 600 nm of 0.2—0.3. Next, 10 µL of cells were added to 1 mL of PBS at pH 7.4 and examined by flow cytometry.</p>
<p>Flow cytometry data was collected on an S3e cell sorter (Bio-Rad, Hercules) using a target flow rate of 2,000 counts/s and collecting 30,000 counts for each sample. A 488-nm laser line was used for excitation at its maximum power setting with amplification settings of 450 (forward scattering, FSC), 350 (side scattering, SSC), and 900 (FL1, 525/30 nm). The cell sorter is calibrated daily for a linear response to sample fluorescence intensity. Acquisition was triggered by forward scattering with a threshold of 3. Data was exported as an FCS file and imported into a custom Python script using FlowCal <span class="citation" data-cites="J6PWaBUu">[<a href="#ref-J6PWaBUu">20</a>]</span>. Following previous methods <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>, one third of samples were selected based on proximity to the peak of FSC-AREA and SSC-HEIGHT in a 2D histogram using the density2d method in FlowCal. The FL1-AREA measurements were used to estimate the mean and variance of GFP distributions for all samples. This was done by estimating the probability density functions in bins distributed equally in logarithmic space and fitting by least squares minimization to a gamma function. We found that this method reduced the influence of low-FL1-AREA events that escaped other gating steps, and which had frequencies that varied for different samples and days (<strong>Figure <a href="#fig:gammaFits">S1</a></strong>). In all plots, the mean fluorescence of a strain harboring a similar plasmid, pZH501, that does not encode a fluorescent protein, was subtracted <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. The script for data analysis as well as all raw FCS data is available in <strong>File S2</strong> and utilized modules from SciPy, NumPy, Matplotlib, and Pandas.</p>
<h3 id="microscopy">Microscopy</h3>
<p>For microscopy, cells were grown similarly to the protocol for flow cytometry, except that overnight cultures were diluted 1:100 and imaged after 2—4 hours of growth with induction. For mRNA imaging, cells were grown in M9 media supplemented with 1X MEM Amino Acids (M9A) or M9A additionally supplemented with 1% SOB media. For 2-plasmid imaging, cells were grown in EZ Rich media. Cells were imaged on agarose gel pads (3% BP165-25, Fisher Bio-Reagents) diluted in M9A (for mRNA imaging) or PBS (for 2-plasmid imaging). For mRNA imaging, the microscope sample chamber was maintained at 30 °C. All imaging was done on a Leica DMI6000 inverted microscope using illumination from a Leica EL6000 source (at various intensities ensuring minimal photobleaching during acquisition), fluorescence filter cubes (Leica GFP ET, a custom filter set with Semrock filters FF01-561/13, FF02-616/73, and DI02-R561, or the Semrock LF514-B filter set), a 100x/1.46 a-plan apochromat oil immersion objective, and an Evolve 512 EM-CCD camera (Photometrics). Images were prepared using Fiji <span class="citation" data-cites="1Xy8hvPg">[<a href="#ref-1Xy8hvPg">21</a>]</span>, with linear scaling and maintaining minimum and maximum intensity values for all comparable images.</p>
<p>To analyze microscopy data for 2-plasmid imaging, 100 cells from each sample were manually segmented using the selection brush tool in Fiji with a width of 10 pixels. This selection was used to extract the mean green and red intensities (proportional to the concentration of GFPmut2 and mScarlet-I molecules in the cell, respectively) in images of the same areas. For each image, the mean background intensity was also measured from a region containing no cells, which was subtracted from each single-cell data point. The average fluorescence level was calculated for each condition, and normalized by the highest average value for each color (20,500 counts/pixel for green and 10,534 counts/pixel for red).</p>
<h2 id="results">Results</h2>
<h3 id="moving-bicistronic-autoregulatory-construct-to-a-compatible-plasmid-backbone">Moving bicistronic autoregulatory construct to a compatible plasmid backbone</h3>
<p>The first step in creating a low-noise system for tuning expression of two genes is to establish that a previously characterized bicistronic autoregulatory circuit functions well in a compatible plasmid backbone. In this expression system, GFP and TetR are expressed bicistronically from the TetR-repressible promoter P<sub>LtetO-1</sub> and expression is induced by the addition of ATc <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. This system was shown to have low noise and a linearized dose response compared to a system in which TetR is constitutively expressed. We moved the system from a plasmid with a p15A replicon conferring ampicillin resistance to a lower-copy-number plasmid with a pSC101 replicon conferring spectinomycin resistance <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">10</a>]</span>. The p15A and pSC101 replicons have been used together in multiplasmid systems <span class="citation" data-cites="15TQ1MvYV">[<a href="#ref-15TQ1MvYV">22</a>]</span>.</p>
<p>GFP expression mean and noise were characterized from low to high levels of induction by flow cytometry. <strong>Figure <a href="#fig:pZH509-pJS101">1</a></strong> shows that pJS101 induces at similar ATc concentrations as pZH509, with the change to the lower-copy pSC101 backbone resulting in a 58% drop in mean expression levels at a wide range of ATc concentrations. For a similar expression system in the absence of autoregulated TetR expression, moving the P<sub>LtetO-1</sub> promoter from a p15A to a pSC101 backbone resulted in an 87% drop in expression <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>. A smaller change is expected in our experiment, since negative autoregulation will provide dosage compensation, just as autoregulation can reduce noise in plasmid copy number <span class="citation" data-cites="GcRoYBF2 Qz6zNiXK S7KRThVS">[<a href="#ref-GcRoYBF2">23</a>,<a href="#ref-Qz6zNiXK">24</a>,<a href="#ref-S7KRThVS">25</a>]</span>.</p>
<figure>
<img src="images/Fig1_pZH509_pJS101.png" alt="Figure 1: Influence of plasmid backbone on induction of TetR expression system. Mean single-cell GFP fluorescence for pZH509 (blue) and pJS101 (orange) plasmids as a function of ATc concentration. Error bars are 1 standard error." id="fig:pZH509-pJS101" style="width:89mm" /><figcaption><span>Figure 1:</span> <strong>Influence of plasmid backbone on induction of TetR expression system.</strong> Mean single-cell GFP fluorescence for pZH509 (blue) and pJS101 (orange) plasmids as a function of ATc concentration. Error bars are 1 standard error.</figcaption>
</figure>
<h3 id="alternative-regulatory-constructs-with-laci-replacing-tetr">Alternative regulatory constructs with LacI replacing TetR</h3>
<p>We hypothesized that replacing P<sub>LtetO-1</sub> with the inducible promoter P<sub>LlacO-1</sub>, with similar characteristics <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>, and replacing TetR with LacI might result in a similarly useful expression system that could be tuned independently. However, regulatory parameters for TetR and LacI vary significantly. TetR binds <em>tetO2</em> more strongly than LacI binds <em>lacO1</em> (approximately 0.5—1.0 kcal/mol difference in binding energy <span class="citation" data-cites="cpANz6cV UMSODjob">[<a href="#ref-cpANz6cV">26</a>,<a href="#ref-UMSODjob">27</a>]</span> for a single site, with 2 tandem sites in our constructs). And, TetR binds ATc much more strongly than LacI binds IPTG <span class="citation" data-cites="Ls5d7x4o WxFVKQqb">[<a href="#ref-Ls5d7x4o">28</a>,<a href="#ref-WxFVKQqb">29</a>]</span>.</p>
<p>We first characterized induction of GFP expression in MG1655 cells harboring IPTG-inducible pJS102 by flow cytometry. <strong>Figure <a href="#fig:pJS102-mean-noise">2</a>a</strong> shows an induction range of almost 2 orders of magnitude from 0 to 1250 µM IPTG, with very good reproducibility of induction levels in 3 independent experiments. Previous experiments with the TetR-based system showed a large jump in expression going from 0 nM to 0.5 nM ATc <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. This effect is not seen for pJS102, suggesting that switching from TetR:ATc to LacI:IPTG improves the dynamic range of achievable induction levels to a small extent.</p>
<figure>
<img src="images/Fig2_pJS102_mean_noise.png" alt="Figure 2: Characterizing mean expression levels and noise for different gene expression systems. (a) Mean GFP expression for pJS102 with different IPTG concentrations. Zero data point is plotted separately to fit on logarithmic scale. Error bars 1 standard error. (b) GFP expression noise (CV2) as a function of mean for pZH509, pJS101, and pJS102. GFP mean increases monotonically with inducer concentration. Error bars 1 standard error." id="fig:pJS102-mean-noise" style="width:183mm" /><figcaption><span>Figure 2:</span> <strong>Characterizing mean expression levels and noise for different gene expression systems.</strong> (<strong>a</strong>) Mean GFP expression for pJS102 with different IPTG concentrations. Zero data point is plotted separately to fit on logarithmic scale. Error bars 1 standard error. (<strong>b</strong>) GFP expression noise (CV<sup>2</sup>) as a function of mean for pZH509, pJS101, and pJS102. GFP mean increases monotonically with inducer concentration. Error bars 1 standard error.</figcaption>
</figure>
<p>Next, we compared noise in protein expression, with the concern that the <em>lac</em> operon present in the MG1655 host strain could lead to all-or-none expression at intermediate IPTG concentrations <span class="citation" data-cites="eUtnfAD6">[<a href="#ref-eUtnfAD6">30</a>]</span>. However, <strong>Figure <a href="#fig:pJS102-mean-noise">2</a>b</strong> shows low noise in GFP expression at all IPTG concentrations, with noise levels comparable to pZH509 and pJS102 at the same mean GFP levels. Note that apparently high noise at very low expression is partially due to measurement noise (see Methods), and, at any rate, is much lower than noise when expression is regulated by a constitutively expressed repressor <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">1</a>]</span>. Here, we also note that noise for pJS101, with its lower-copy-number pSC101 replicon, is lower than that for pZH509 or pJS102 at similar expression levels. This suggests that incorporating this construct into the chromosome, where copy number is more tightly regulated, may lead to a further noise reduction.</p>
<p>We note that side scattering was weakly correlated with fluorescence, and thus with cell size, so gating for scattering modestly reduces measured noise in fluorescence intensity. However, we are comparing to an “extrinsic noise limit” determined from measurements of cell fluorescence divided by cell area <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>, which effectively does the same thing. In practice, we observe slightly lower noise measurements for GFP concentration in fluorescence microscopy images compared to total GFP fluorescence in the gated flow cytometry sample for similarly induced strains. This difference was more significant for very-low-expression conditions, and noise in conditions where GFP fluorescence distributions significantly overlap with ungated background events (GFP intensity less than 10<sup>4</sup> in <strong>Figure <a href="#fig:gammaFits">S1</a></strong>) is somewhat overestimated.</p>
<h3 id="using-the-new-induction-system-for-detection-of-single-mrna-in-living-e.-coli">Using the new induction system for detection of single mRNA in living <em>E. coli</em></h3>
<p>Recently, an improved method for detection of mRNA by local enrichment of fluorescent, RNA-binding proteins was reported in <em>S. cerevisiae</em> <span class="citation" data-cites="82diXnad">[<a href="#ref-82diXnad">4</a>]</span>. This reduced the aggregation of mRNAs bound by the bacteriophage MS2 coat protein, which has also caused mRNA immortalization that has limited experiments in <em>E. coli</em> to observing transcription just after induction <span class="citation" data-cites="TvXapFFQ">[<a href="#ref-TvXapFFQ">31</a>]</span>. We hypothesized that aggregation could be reduced by reducing the expression levels of both mRNA and mRNA-binding proteins, and by having low cell-to-cell variation in expression. We developed a strain in which mRNA molecules encoding mVenus-Cro and also including 24 tandem repeats of the binding sequence for the PP7 coat protein (PP7cp) <span class="citation" data-cites="1GFLV3s4q">[<a href="#ref-1GFLV3s4q">15</a>]</span> were constitutively expressed at low levels (less than 1 molecule per cell). Plasmid pJS102 was used as a template to develop a fluorescent, IPTG-inducible reporter of expression, PP7cp-SYFP2.</p>
<p>We tested the utility of this expression system for tuning low-noise gene expression in different growth conditions. In previous experiments, we expressed the RNA-binding protein from a constitutive promoter integrated into the chromosome; this required long cycles of optimization every time a parameter was changed (growth media, temperature, fluorescent protein sequence) that changed protein expression levels. <strong>Figure <a href="#fig:pZH713-optimization">3</a>a</strong> shows that single-molecule mRNA detection was optimal at 100 µM IPTG in partially rich media. We note the absence of pole-localized fluorescent spots that characterize mRNA aggregation <span class="citation" data-cites="uQeM79z">[<a href="#ref-uQeM79z">32</a>]</span>, and we observed reasonable mRNA lifetimes of a few minutes in timelapse imaging. We moved to minimal media hoping to find a growth condition with lower mRNA expression levels, but found that 100 µM IPTG gave a background of unbound PP7cp-SYFP2 molecules that often made it impossible to detect mRNA spots. <strong>Figure <a href="#fig:pZH713-optimization">3</a>b</strong> shows how the IPTG-inducible expression system made it simple to quickly scan several different PP7cp-SYFP induction conditions and identify 10—20 µM IPTG as an optimal growth condition.</p>
<p>Lastly, we note that the strain used for mRNA imaging has its entire <em>lac</em> operon replaced by the synthetic construct. Thus, this expression system works well both in the presence and absence of the <em>lac</em> operon.</p>
<figure>
<img src="images/Fig3_pZH713_optimization.png" alt="Figure 3: Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions. (a) PP7cp-SYFP2 is induced with 100 µM IPTG to detect single mVenus-Cro mRNA molecules in supplemented media and minimal media conditions; in minimal media there is too high a PP7cp-SYFP expression level to see single mRNA spots above background. Scale bar 4 µm. (b) Using the pJS102 expression system, an optimal range of 10—20 µM IPTG is quickly identified for single mRNA detection in minimal media. Scale bar 2 µM." id="fig:pZH713-optimization" style="width:89mm" /><figcaption><span>Figure 3:</span> <strong>Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.</strong> (<strong>a</strong>) PP7cp-SYFP2 is induced with 100 µM IPTG to detect single mVenus-Cro mRNA molecules in supplemented media and minimal media conditions; in minimal media there is too high a PP7cp-SYFP expression level to see single mRNA spots above background. Scale bar 4 µm. (<strong>b</strong>) Using the pJS102 expression system, an optimal range of 10—20 µM IPTG is quickly identified for single mRNA detection in minimal media. Scale bar 2 µM.</figcaption>
</figure>
<h3 id="independent-tunable-expression-of-two-genes">Independent, tunable expression of two genes</h3>
<p>Lastly, we tested whether ATc-inducible and IPTG-inducible plasmids could be combined to achieve low-noise expression of two genes into the same cell. We replaced GFPmut2 in pJS102 with the fast-maturing RFP mScarlet-I <span class="citation" data-cites="zj8Ydkzo">[<a href="#ref-zj8Ydkzo">18</a>]</span> to create the plasmid pDG101. This plasmid was co-transformed with pJS101 into <em>E. coli</em> MG1655 and green and red fluorescence were compared at different combinations of ATc and IPTG concentrations. <strong>Figure <a href="#fig:pJS101-pDG101">4</a>a</strong> shows that pJS101 induction by ATc was unaffected by pDG101 induction by IPTG, and that all conditions gave low noise in GFP concentration. <strong>Figure <a href="#fig:pJS101-pDG101">4</a>b</strong> shows that mScarlet-I expression from pDG101 was similarly unaffected by the level of pJS101 induction by ATc. Thus, independent, tunable expression of two genes can be achieved by replacing GFPmut2 in pJS101 and pJS102 with other genes of interest and co-transforming the plasmids into <em>E. coli</em>.</p>
<figure>
<img src="images/Fig4_pJS101_pDG101.png" alt="Figure 4: Independence of induction of TetR and LacI systems. MG1655 cells harboring pJS101 (ATc-inducible expression of GFP) and pDG101 (IPTG-inducible expression of mScarlet-I) were grown with different combinations of IPTG and ATc concentrations. (a) GFP fluorescence shows no influence of IPTG on ATc-induced expression of GFP. (b) No influence of ATc on IPTG-induced expression of mScarlet-I. Scale bar 3 µm. (c) Average single-cell GFP (green) and mScarlet-I (red) fluoresence, normalized by the maximum average fluorescence for each color. Solid lines: 10 nM ATc. Dashed lines: 50 nM ATc." id="fig:pJS101-pDG101" style="width:89mm" /><figcaption><span>Figure 4:</span> <strong>Independence of induction of TetR and LacI systems.</strong> MG1655 cells harboring pJS101 (ATc-inducible expression of GFP) and pDG101 (IPTG-inducible expression of mScarlet-I) were grown with different combinations of IPTG and ATc concentrations. (<strong>a</strong>) GFP fluorescence shows no influence of IPTG on ATc-induced expression of GFP. (<strong>b</strong>) No influence of ATc on IPTG-induced expression of mScarlet-I. Scale bar 3 µm. (<strong>c</strong>) Average single-cell GFP (green) and mScarlet-I (red) fluoresence, normalized by the maximum average fluorescence for each color. Solid lines: 10 nM ATc. Dashed lines: 50 nM ATc.</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>We expect that the expression plasmids introduced here will be useful for diverse applications in molecular biology. Expression and purification of heteromeric protein complexes could be improved by stoichiometric production of their components, mimicking proportional synthesis in natural systems <span class="citation" data-cites="12aHdKu6v">[<a href="#ref-12aHdKu6v">33</a>]</span>. Additionally, low-noise expression can improve protein production yields <span class="citation" data-cites="5dC8qsZ9">[<a href="#ref-5dC8qsZ9">34</a>]</span>. These systems could also be used in synthetic biology applications where yields can be improved by sequential induction of different components with low cell-to-cell variability. The capacity for low-noise expression at very low expression levels makes them particularly promising for single-molecule imaging experiments or for recombinant expression of low-copy-number components with low cell-to-cell variation to reproduce chromosomal expression levels.</p>
<p>Using modern molecular cloning techniques, it is simple to replace GFPmut2 in pJS101 and pJS102 with genes of interest. The apparent insensitivity of this circuit to regulatory parameters (DNA and inducer binding affinities) suggests that it can be easily extended to a third repressor-based expression system. Further, additional ribosome binding sites can be added to the bicistronic operon to express additional components.</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JS, DG and ZH designed experiments and performed experiments. JS, SL, DG, and ZH performed molecular cloning. ZH and JS analyzed data and wrote the paper. JS, SL, DG and ZH edited and approved the manuscript. ZH supervised the project.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>This work was financially supported by: Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) funded by FEDER funds through COMPETE2020—Programa Operacional Competitividade e Internacionalização (POCI), by national funds through FCT—Fundação para a Ciência e a Tecnologia, and through a joint research agreement with the Okinawa Institute of Science and Technology (OIST). This manuscript was composed and edited using Manubot <span class="citation" data-cites="1GGGHdsew">[<a href="#ref-1GGGHdsew">35</a>]</span>.</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-TlAItren">
<p>1. <strong>A plasmid-based Escherichia coli gene expression system with cell-to-cell variation below the extrinsic noise limit.</strong><br />
Zach Hensel<br />
<em>PloS one</em> (2017-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084263" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/29084263</a></p>
</div>
<div id="ref-oD6kpSs3">
<p>2. <strong>Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells</strong><br />
Y. Taniguchi, P. J. Choi, G.-W. Li, H. Chen, M. Babu, J. Hearn, A. Emili, X. S. Xie<br />
<em>Science</em> (2010-07-29) <a href="https://doi.org/10.1126/science.1188308" class="uri">https://doi.org/10.1126/science.1188308</a></p>
</div>
<div id="ref-x1Hd2npA">
<p>3. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular Systems Biology</em> (2006-08-01) <a href="https://doi.org/10.1038/msb4100081" class="uri">https://doi.org/10.1038/msb4100081</a></p>
</div>
<div id="ref-82diXnad">
<p>4. <strong>An improved MS2 system for accurate reporting of the mRNA life cycle.</strong><br />
Evelina Tutucci, Maria Vera, Jeetayu Biswas, Jennifer Garcia, Roy Parker, Robert H Singer<br />
<em>Nature methods</em> (2017-11-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29131164" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/29131164</a></p>
</div>
<div id="ref-dc6GAnhL">
<p>5. <strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong><br />
Daniel G Gibson, Lei Young, Ray-Yuan Chuang, J Craig Venter, Clyde A Hutchison, Hamilton O Smith<br />
<em>Nature Methods</em> (2009-04-12) <a href="https://doi.org/10.1038/nmeth.1318" class="uri">https://doi.org/10.1038/nmeth.1318</a></p>
</div>
<div id="ref-NCWOk1rp">
<p>6. <strong>Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</strong><br />
R Lutz, H Bujard<br />
<em>Nucleic acids research</em> (1997-03-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/9092630</a></p>
</div>
<div id="ref-13TJF2gcl">
<p>7. <strong>FACS-optimized mutants of the green fluorescent protein (GFP).</strong><br />
BP Cormack, RH Valdivia, S Falkow<br />
<em>Gene</em> (1996) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8707053" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8707053</a></p>
</div>
<div id="ref-Q1a7tU8">
<p>8. <strong>Nucleotide sequence of the repressor gene of the TN10 tetracycline resistance determinant</strong><br />
Kathleen Postle, Toai T. Nguyen, Kevin P. Bertrand<br />
<em>Nucleic Acids Research</em> (1984) <a href="https://doi.org/10.1093/nar/12.12.4849" class="uri">https://doi.org/10.1093/nar/12.12.4849</a></p>
</div>
<div id="ref-xjKSoV9v">
<p>9. <strong>Intrinsic termination of T7 RNA polymerase mediated by either RNA or DNA.</strong><br />
L Hartvig, J Christiansen<br />
<em>The EMBO journal</em> (1996-09-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8887568" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8887568</a></p>
</div>
<div id="ref-5CRbRLYC">
<p>10. <strong>A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.</strong><br />
G Churchward, D Belin, Y Nagamine<br />
<em>Gene</em> (1984-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6098521" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/6098521</a></p>
</div>
<div id="ref-YoMCfbca">
<p>11. <strong>Isolation of the lac repressor.</strong><br />
W Gilbert, B Müller-Hill<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (1966-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16591435" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16591435</a></p>
</div>
<div id="ref-A1kNMUKB">
<p>12. <strong>Cell-cycle-synchronized, oscillatory expression of a negatively autoregulated gene in E. coli</strong><br />
Zach Hensel, Tatiana T. Marquez-Lago<br />
<em>arXiv</em> (2015-06-29) <a href="https://arxiv.org/abs/1506.08596v1" class="uri">https://arxiv.org/abs/1506.08596v1</a></p>
</div>
<div id="ref-RBKXt78E">
<p>13. <strong>One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.</strong><br />
KA Datsenko, BL Wanner<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (2000-06-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/10829079</a></p>
</div>
<div id="ref-pzTydEMA">
<p>14. <strong>Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.</strong><br />
Amin Espah Borujeni, Anirudh S Channarasappa, Howard M Salis<br />
<em>Nucleic acids research</em> (2013-11-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24234441" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/24234441</a></p>
</div>
<div id="ref-1GFLV3s4q">
<p>15. <strong>Real-time observation of transcription initiation and elongation on an endogenous yeast gene.</strong><br />
Daniel R Larson, Daniel Zenklusen, Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Science (New York, N.Y.)</em> (2011-04-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21512033" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/21512033</a></p>
</div>
<div id="ref-XuRR8Zmt">
<p>16. <strong>Cyan and yellow super fluorescent proteins with improved brightness, protein folding, and FRET Förster radius.</strong><br />
Gert-Jan Kremers, Joachim Goedhart, Erik B van Munster, Theodorus WJ Gadella<br />
<em>Biochemistry</em> (2006-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16716067" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16716067</a></p>
</div>
<div id="ref-EObeLKJ1">
<p>17. <strong>Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells.</strong><br />
Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Biophysical journal</em> (2012-06-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22735544" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/22735544</a></p>
</div>
<div id="ref-zj8Ydkzo">
<p>18. <strong>Characterization of a spectrally diverse set of fluorescent proteins as FRET acceptors for mTurquoise2.</strong><br />
Marieke Mastop, Daphne S Bindels, Nathan C Shaner, Marten Postma, Theodorus WJ Gadella, Joachim Goedhart<br />
<em>Scientific reports</em> (2017-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28931898" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/28931898</a></p>
</div>
<div id="ref-q8xLcfYV">
<p>19. <strong>Zach Hensel Lab Plasmids</strong><br />
Zach Hensel, João Silva<br />
<em>AddGene</em> (2019) <a href="https://www.addgene.org/Zach_Hensel/" class="uri">https://www.addgene.org/Zach_Hensel/</a></p>
</div>
<div id="ref-J6PWaBUu">
<p>20. <strong>FlowCal: A User-Friendly, Open Source Software Tool for Automatically Converting Flow Cytometry Data from Arbitrary to Calibrated Units.</strong><br />
Sebastian M Castillo-Hair, John T Sexton, Brian P Landry, Evan J Olson, Oleg A Igoshin, Jeffrey J Tabor<br />
<em>ACS synthetic biology</em> (2016-05-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27110723" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/27110723</a></p>
</div>
<div id="ref-1Xy8hvPg">
<p>21. <strong>Fiji: an open-source platform for biological-image analysis.</strong><br />
Johannes Schindelin, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, Curtis Rueden, Stephan Saalfeld, Benjamin Schmid, … Albert Cardona<br />
<em>Nature methods</em> (2012-06-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/22743772</a></p>
</div>
<div id="ref-15TQ1MvYV">
<p>22. <strong>BglBrick vectors and datasheets: A synthetic biology platform for gene expression.</strong><br />
Taek Soon Lee, Rachel A Krupa, Fuzhong Zhang, Meghdad Hajimorad, William J Holtz, Nilu Prasad, Sung Kuk Lee, Jay D Keasling<br />
<em>Journal of biological engineering</em> (2011-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21933410" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/21933410</a></p>
</div>
<div id="ref-GcRoYBF2">
<p>23. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis.</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular systems biology</em> (2006-08-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16883354" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16883354</a></p>
</div>
<div id="ref-Qz6zNiXK">
<p>24. <strong>Engineering stability in gene networks by autoregulation.</strong><br />
A Becskei, L Serrano<br />
<em>Nature</em> (2000-06-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10850721" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/10850721</a></p>
</div>
<div id="ref-S7KRThVS">
<p>25. <strong>Summing up the noise in gene networks.</strong><br />
Johan Paulsson<br />
<em>Nature</em> (2004-01-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14749823" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/14749823</a></p>
</div>
<div id="ref-cpANz6cV">
<p>26. <strong>Two mutations in the tetracycline repressor change the inducer anhydrotetracycline to a corepressor.</strong><br />
Annette Kamionka, Joanna Bogdanska-Urbaniak, Oliver Scholz, Wolfgang Hillen<br />
<em>Nucleic acids research</em> (2004-02-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14764926" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/14764926</a></p>
</div>
<div id="ref-UMSODjob">
<p>27. <strong>The three operators of the lac operon cooperate in repression.</strong><br />
S Oehler, ER Eismann, H Krämer, B Müller-Hill<br />
<em>The EMBO journal</em> (1990-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2182324" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/2182324</a></p>
</div>
<div id="ref-Ls5d7x4o">
<p>28. <strong>Tet repressor induction without Mg2+.</strong><br />
O Scholz, P Schubert, M Kintrup, W Hillen<br />
<em>Biochemistry</em> (2000-09-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10978179" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/10978179</a></p>
</div>
<div id="ref-WxFVKQqb">
<p>29. <strong>Induction of the lac promoter in the absence of DNA loops and the stoichiometry of induction.</strong><br />
Stefan Oehler, Siegfried Alberti, Benno Müller-Hill<br />
<em>Nucleic acids research</em> (2006-01-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16432263" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16432263</a></p>
</div>
<div id="ref-eUtnfAD6">
<p>30. <strong>ENZYME INDUCTION AS AN ALL-OR-NONE PHENOMENON.</strong><br />
A Novick, M Weiner<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (1957-07-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16590055" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16590055</a></p>
</div>
<div id="ref-TvXapFFQ">
<p>31. <strong>Real-time kinetics of gene activity in individual bacteria.</strong><br />
Ido Golding, Johan Paulsson, Scott M Zawilski, Edward C Cox<br />
<em>Cell</em> (2005-12-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16360033" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16360033</a></p>
</div>
<div id="ref-uQeM79z">
<p>32. <strong>Spatial organization of the flow of genetic information in bacteria.</strong><br />
Paula Montero Llopis, Audrey F Jackson, Oleksii Sliusarenko, Ivan Surovtsev, Jennifer Heinritz, Thierry Emonet, Christine Jacobs-Wagner<br />
<em>Nature</em> (2010-06-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20562858" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/20562858</a></p>
</div>
<div id="ref-12aHdKu6v">
<p>33. <strong>Production of Protein-Complex Components Is Stoichiometric and Lacks General Feedback Regulation in Eukaryotes.</strong><br />
James C Taggart, Gene-Wei Li<br />
<em>Cell systems</em> (2018-12-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30553725" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/30553725</a></p>
</div>
<div id="ref-5dC8qsZ9">
<p>34. <strong>Comparative Single-Cell Analysis of Different E. coli Expression Systems during Microfluidic Cultivation.</strong><br />
Dennis Binder, Christopher Probst, Alexander Grünberger, Fabienne Hilgers, Anita Loeschcke, Karl-Erich Jaeger, Dietrich Kohlheyer, Thomas Drepper<br />
<em>PloS one</em> (2016-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27525986" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/27525986</a></p>
</div>
<div id="ref-1GGGHdsew">
<p>35. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>Manubot Preprint</em> (2018) <a href="https://greenelab.github.io/meta-review/" class="uri">https://greenelab.github.io/meta-review/</a></p>
</div>
</div>
<h2 id="supplementary-material" class="page_break_before">Supplementary Material</h2>
<figure>
<img src="images/FigureS1_gammaFits.png" alt="Figure S1: Reproducibility of low-noise expression in 3 independent experiments. Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples." id="fig:gammaFits" data-tag="S1" style="width:89mm" /><figcaption><span>Figure S1:</span> <strong>Reproducibility of low-noise expression in 3 independent experiments.</strong> Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples.</figcaption>
</figure>
<p><strong>Supplementary File S2</strong>: Raw flow cytometry data, Python scripts required to reproduce Figures <a href="#fig:pZH509-pJS101">1</a> and <a href="#fig:pJS102-mean-noise">2</a>, DNA sequences, and explanatory text files are available as a compressed archive at Zenodo.</p>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
