<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="João Silva" />
  <meta name="author" content="Soraia Lopes" />
  <meta name="author" content="Diogo Grilo" />
  <meta name="author" content="Zach Hensel" />
  <meta name="dcterms.date" content="2018-12-27" />
  <meta name="keywords" content="gene expression, synthetic biology, transcription" />
  <title>Plasmids for independently tunable, low-noise gene expression</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css">
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header>
<h1 class="title">Plasmids for independently tunable, low-noise gene expression</h1>
</header>
<p><small><em> This manuscript (<a href="https://zach-hensel.github.io/low-noise-manuscript/v/14fc71df64df235045f3f175976060753b9600f3/">permalink</a>) was automatically generated from <a href="https://github.com/zach-hensel/low-noise-manuscript/tree/14fc71df64df235045f3f175976060753b9600f3">zach-hensel/low-noise-manuscript@14fc71d</a> on December 27, 2018. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>João Silva</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Soraia Lopes</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Diogo Grilo</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Zach Hensel</strong><br> <a href="mailto:zach.hensel@itqb.unl.pt">zach.hensel@itqb.unl.pt</a> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-4348-6229">0000-0002-4348-6229</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/zach-hensel">zach-hensel</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/alchemytoday">alchemytoday</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal · Funded by Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Some microbiology experiments and biotechnology applications can be improved if it is possible to tune the expression of two different genes at the same time with cell-to-cell variation at or below the level of genes constitutively expressed from the chromosome (the “extrinsic noise limit”). This was recently achieved for a single gene by exploiting negative autoregulation by the tetracycline repressor (TetR), and bicistronic gene expression to reduce gene expression noise. We report new plasmids that use the same principles to achieve simultaneous, low-noise expression for two genes. The TetR system was moved to a compatible plasmid backbone, and a system based on the lactose repressor (LacI) was found to also exhibit gene expression noise below the extrinsic noise limit. We characterized gene expression mean and noise across the range of induction levels for these strains, applied the LacI system to tune expression for single-molecule mRNA detection in two different growth conditions, and showed that two plasmids can be co-transformed to independently tune expression of two different genes.</p>
<h2 id="introduction" class="page_break_before">Introduction</h2>
<p>We recently reported the development of a plasmid-based gene-expression system in which a gene of interest is expressed bicistronically with the tetracycline repressor (TetR) <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span>. Using this gene expression system, cell-to-cell is below the “extrinsic noise limit” observed for genes expressed from the chromosome <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>. When TetR and GFP are expressed bicistronically, GFP induction and gene expression noise is similar to that observed for a TetR-GFP fusion protein with autoregulation <span class="citation" data-cites="x1Hd2npA">[<a href="#ref-x1Hd2npA">3</a>]</span>. Compared to induction of gene expression under the control of a constitutively expressed transcriptional repressor, the inducer dose-response is relatively linearized, and gene expression noise is much lower at intermediate induction levels.</p>
<p>Why this would it be useful to do this for two different genes at the same time? Example for stoichiometric production of a protein complex Expressing genes at their native levels … e.g. for generating chromosome deletions</p>
<p>Figure 1: Regulatory constructs used in this study. (a) pZH509, (b) pJS102, (c) pJS103, (d) pJS23103</p>
<h2 id="methods">Methods</h2>
<h3 id="strain-construction">Strain construction</h3>
<p>All plasmids were constructed using isothermal assembly <span class="citation" data-cites="dc6GAnhL">[<a href="#ref-dc6GAnhL">4</a>]</span> of fragments generated by PCR or double stranded DNA synthesis (IDT) and transformed Top10 E. coli cells (IBA #5-1600-020). Transformants were screened by colony PCR and verified by sequencing. Sequence maps are included as supplementary files and plasmids will be made available at AddGene after deposit and screening <span class="citation" data-cites="q8xLcfYV">[<a href="#ref-q8xLcfYV">5</a>]</span>. Purified plasmids were transformed into E. coli strain MG1655 by growing 2 mL of culture in SOB media at 37 C to OD600=0.4, washing twice with 1 mL ice-cold water, resuspending in 40 µL water, electroporation of 1–10 ng plasmid with the EC1 setting of a Micropulser (BioRad), and recovering for 1 hour at 37 C in SOC media.</p>
<p>To generate pJS101 with a compatible backbone, plasmid pZH509 <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span> was used as a template to amplify the bicistronic regulatory construct including the P<sub>LtetO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>, GFPmut2 <span class="citation" data-cites="13TJF2gcl">[<a href="#ref-13TJF2gcl">7</a>]</span>, tn10 TetR <span class="citation" data-cites="Q1a7tU8">[<a href="#ref-Q1a7tU8">8</a>]</span> and rrnB T1 transcription terminator <span class="citation" data-cites="xjKSoV9v">[<a href="#ref-xjKSoV9v">9</a>]</span>. This was assembled by isothermal assembly with the backbone from pGB2 <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">10</a>]</span> with the pSC101 origin of replication and spectinomycin resistance to generate plasmid pJS101. Plasmids with the pSC101 and p15a origins of replication have been co-transformed in previous synthetic biology experiments <span class="citation" data-cites="fyyCRa8e">[<a href="#ref-fyyCRa8e">11</a>]</span>.</p>
<p>Plasmids pJS102 and pJS103 were generated by 3-fragment isothermal assembly. Plasmid pZH509 was used as a template both for the vector backbone and for GFPmut2, with non-homologous extensions added to PCR primers to generate the P<sub>LlacO</sub> and P<sub>LlacOsym</sub> promoters. Promoter P<sub>LlacOsym</sub> lacks the central G–C base pair in P<sub>LlacO</sub> LacI binding sites, which are symmetrical <span class="citation" data-cites="swicCLW1">[<a href="#ref-swicCLW1">12</a>]</span> except that the second <em>lacOsym</em> sequence has one base changed to maintain the P<sub>LlacO</sub> promoter –10 sequence. LacI was amplified from <em>E. coli</em> MG1655 by colony PCR. Plasmid pJS23103 was made by isothermal assembly of an inverse PCR product of pZH509, with non-homologous primer extensions adding the weak, constitutive BBa_J23103 promoter between GFPmut2 and TetR <span class="citation" data-cites="ZZUG1lTS">[<a href="#ref-ZZUG1lTS">13</a>]</span>.</p>
<p>Plasmids were co-transformed into MG1655 by electroporation following the above protocol, except with 1 µL each undiluted plasmid (~20–40 ng) and selecting on plates with both spectinomycin and carbenicillin.</p>
<a name="tbl:plasmidList"></a>
<table>
<caption><span>Table 1:</span> Plasmids used in this study. </caption>
<thead>
<tr class="header">
<th>Plasmid</th>
<th>Ori</th>
<th>Promoter</th>
<th>GOI</th>
<th>Repressor Expression</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pZH501</td>
<td>p15a</td>
<td>CI-SNAP-tag</td>
<td><sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span></td>
</tr>
<tr class="even">
<td>pZH509</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span></td>
</tr>
<tr class="odd">
<td>pZH520</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>constitutive</td>
<td><span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span></td>
</tr>
<tr class="even">
<td>pJS101</td>
<td>pSC101</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pJS102</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LlacO</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pJS103</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LlacOsym</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pJS23103</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic/constitutive</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pZH742</td>
<td>pSC101</td>
<td>PP7cp-mNeonGreen</td>
<td>P<sub>LlacO</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pZH740</td>
<td>p15a</td>
<td>mScarlet-I-24xPP7sl</td>
<td>P<sub>LtetO-1</sub></td>
<td>constitutive</td>
<td>This work</td>
</tr>
</tbody>
</table>
<h3 id="cell-growth">Cell Growth</h3>
<p>Cells were grown in overnight cultures at 32 C in M9 minimal media supplemented with 1% SOB media, 50 µg/mL carbenicillin or 50 µg/mL spectinomycin. The next day, cells were diluted 1:100 or more (if multiple experiments were to be done sequentially over several hours) in the same media supplemented with anhydrotetracycline. In this growth condition, doubling time was approximately 30 minutes for MG1655 strains harboring these plasmids.</p>
<h3 id="microscopy">Microscopy</h3>
<p>Microscopy methods and microscopy data analysis methods</p>
<h3 id="flow-cytometry">Flow cytometry</h3>
<p>Flow cytometry methods and data analysis methods</p>
<h2 id="results">Results</h2>
<h3 id="moving-bicistronic-autoregulatory-construct-to-a-compatible-plasmid-backbone">Moving bicistronic autoregulatory construct to a compatible plasmid backbone</h3>
<p>Info and testing for pJS101</p>
<p>Info for pJS102 / pJS103</p>
<p>Figure 2: pZH509 vs pJS101 (a) mean(aTc), (b) noise(mean) flow cytometry</p>
<h3 id="alternative-regulatory-constructs-with-laci-replacing-tetr">Alternative regulatory constructs with LacI replacing TetR</h3>
<p>We hypothesized that replacing P<sub>LtetO-1</sub> with a promoter with similar characteristics and replacing TetR with a …</p>
<p>Figure 3: pZH509 vs pJS102 vs pJS103 (a) mean(aTc), (b) noise(mean), (c) Microscope images at mid induction, (d) noise(mean) from microscope images</p>
<h3 id="increasing-dynamic-range-with-the-addition-of-a-weak-constitutive-promoter">Increasing dynamic range with the addition of a weak constitutive promoter</h3>
<figure>
<img src="images/WeakConstitutivePromoter.png" alt="Figure 1: (A) Mean GFP fluorescence for pJS23103, pZH509 and pZH520 as a function of ATc concentration. The additional weak, constitutive promoter for TetR in pJS23103 results in lower GFP expression at low ATc concentrations. (B) Noise (CV2) as a function of mean GFP expression for the same strains. pJS23103 retains the low gene expression noise characteristic of pZH509. Error bars in both plots are bootstrapped 95% confidence intervals from a single experiment. The mean and variance of the pZH501 strain which does not express GFP was subtracted from each sample." id="fig:weakPromoter" /><figcaption><span>Figure 1:</span> (<strong>A</strong>) Mean GFP fluorescence for pJS23103, pZH509 and pZH520 as a function of ATc concentration. The additional weak, constitutive promoter for TetR in pJS23103 results in lower GFP expression at low ATc concentrations. (<strong>B</strong>) Noise (<em>CV</em><sup>2</sup>) as a function of mean GFP expression for the same strains. pJS23103 retains the low gene expression noise characteristic of pZH509. Error bars in both plots are bootstrapped 95% confidence intervals from a single experiment. The mean and variance of the pZH501 strain which does not express GFP was subtracted from each sample.</figcaption>
</figure>
<p>Figure 4: ZH509 vs pZH520 vs pJS23103 (a) mean(aTc), (b) noise(mean), (c) Microscope images at mid induction, (d) noise(mean) from microscope images</p>
<h3 id="expression-of-two-genes-by-remixing-these-components">Expression of two genes by remixing these components</h3>
<p>Aggregation of mRNA-binding proteins from RNA bacteriophages makes it difficult to use fluorescent fusion proteins to detect single mRNAs <span class="citation" data-cites="cfHwILHp">[<a href="#ref-cfHwILHp">14</a>]</span>. We hypothesized that …</p>
<p><span class="citation" data-cites="ne2ewtkS">[<a href="#ref-ne2ewtkS">15</a>]</span> mNeonGreen <span class="citation" data-cites="H1cK0KNl">[<a href="#ref-H1cK0KNl">16</a>]</span> mScarlet-I <span class="citation" data-cites="oLbIsF1b">[<a href="#ref-oLbIsF1b">17</a>]</span> The Pf3 coat protein is a small membrane-spanning protein with a cytoplasm-facing C terminus.</p>
<h2 id="discussion">Discussion</h2>
<p>Possible to improve pJS23103 with somewhat stronger promoter?</p>
<p>Possible to extend to 3 genes? Possible using additional repressor, pMB1 origin of replication, and different antibiotic selection.</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JS and ZH designed experiments, performed experiments, analyzed data, and wrote the paper. ZH supervised the project.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>This work was financially supported by: Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) funded by FEDER funds through COMPETE2020—Programa Operacional Competitividade e Internacionalização (POCI), by national funds through FCT—Fundação para a Ciência e a Tecnologia, and through a joint research agreement with the Okinawa Institute of Science and Technology (OIST).</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-RB57dahw">
<p>1. <strong>A plasmid-based Escherichia coli gene expression system with cell-to-cell variation below the extrinsic noise limit</strong><br />
Zach Hensel<br />
<em>PLOS ONE</em> (2017-10-30) <a href="https://doi.org/10.1371/journal.pone.0187259" class="uri">https://doi.org/10.1371/journal.pone.0187259</a></p>
</div>
<div id="ref-oD6kpSs3">
<p>2. <strong>Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells</strong><br />
Y. Taniguchi, P. J. Choi, G.-W. Li, H. Chen, M. Babu, J. Hearn, A. Emili, X. S. Xie<br />
<em>Science</em> (2010-07-29) <a href="https://doi.org/10.1126/science.1188308" class="uri">https://doi.org/10.1126/science.1188308</a></p>
</div>
<div id="ref-x1Hd2npA">
<p>3. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular Systems Biology</em> (2006-08-01) <a href="https://doi.org/10.1038/msb4100081" class="uri">https://doi.org/10.1038/msb4100081</a></p>
</div>
<div id="ref-dc6GAnhL">
<p>4. <strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong><br />
Daniel G Gibson, Lei Young, Ray-Yuan Chuang, J Craig Venter, Clyde A Hutchison, Hamilton O Smith<br />
<em>Nature Methods</em> (2009-04-12) <a href="https://doi.org/10.1038/nmeth.1318" class="uri">https://doi.org/10.1038/nmeth.1318</a></p>
</div>
<div id="ref-q8xLcfYV">
<p>5. <a href="https://www.addgene.org/Zach_Hensel/" class="uri">https://www.addgene.org/Zach_Hensel/</a></p>
</div>
<div id="ref-NCWOk1rp">
<p>6. <strong>Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</strong><br />
R Lutz, H Bujard<br />
<em>Nucleic acids research</em> (1997-03-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/9092630</a></p>
</div>
<div id="ref-13TJF2gcl">
<p>7. <strong>FACS-optimized mutants of the green fluorescent protein (GFP).</strong><br />
BP Cormack, RH Valdivia, S Falkow<br />
<em>Gene</em> (1996) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8707053" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8707053</a></p>
</div>
<div id="ref-Q1a7tU8">
<p>8. <strong>Nucleotide sequence of the repressor gene of the TN10 tetracycline resistance determinant</strong><br />
Kathleen Postle, Toai T. Nguyen, Kevin P. Bertrand<br />
<em>Nucleic Acids Research</em> (1984) <a href="https://doi.org/10.1093/nar/12.12.4849" class="uri">https://doi.org/10.1093/nar/12.12.4849</a></p>
</div>
<div id="ref-xjKSoV9v">
<p>9. <strong>Intrinsic termination of T7 RNA polymerase mediated by either RNA or DNA.</strong><br />
L Hartvig, J Christiansen<br />
<em>The EMBO journal</em> (1996-09-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8887568" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8887568</a></p>
</div>
<div id="ref-5CRbRLYC">
<p>10. <strong>A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.</strong><br />
G Churchward, D Belin, Y Nagamine<br />
<em>Gene</em> (1984-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6098521" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/6098521</a></p>
</div>
<div id="ref-fyyCRa8e">
<p>11. <strong>BglBrick vectors and datasheets: A synthetic biology platform for gene expression</strong><br />
Taek Lee, Rachel A Krupa, Fuzhong Zhang, Meghdad Hajimorad, William J Holtz, Nilu Prasad, Sung Lee, Jay D Keasling<br />
<em>Journal of Biological Engineering</em> (2011) <a href="https://doi.org/10.1186/1754-1611-5-12" class="uri">https://doi.org/10.1186/1754-1611-5-12</a></p>
</div>
<div id="ref-swicCLW1">
<p>12. <strong>A perfectly symmetric lac operator binds the lac repressor very tightly.</strong><br />
JR Sadler, H Sasmor, JL Betz<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (1983-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6316325" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/6316325</a></p>
</div>
<div id="ref-ZZUG1lTS">
<p>13. <a href="http://parts.igem.org/Promoters/Catalog/Anderson" class="uri">http://parts.igem.org/Promoters/Catalog/Anderson</a></p>
</div>
<div id="ref-cfHwILHp">
<p>14. <strong>Spatial organization of the flow of genetic information in bacteria</strong><br />
Paula Montero Llopis, Audrey F. Jackson, Oleksii Sliusarenko, Ivan Surovtsev, Jennifer Heinritz, Thierry Emonet, Christine Jacobs-Wagner<br />
<em>Nature</em> (2010-06-20) <a href="https://doi.org/10.1038/nature09152" class="uri">https://doi.org/10.1038/nature09152</a></p>
</div>
<div id="ref-ne2ewtkS">
<p>15. <strong>A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum</strong><br />
Nathan C Shaner, Gerard G Lambert, Andrew Chammas, Yuhui Ni, Paula J Cranfill, Michelle A Baird, Brittney R Sell, John R Allen, Richard N Day, Maria Israelsson, … Jiwu Wang<br />
<em>Nature Methods</em> (2013-03-24) <a href="https://doi.org/10.1038/nmeth.2413" class="uri">https://doi.org/10.1038/nmeth.2413</a></p>
</div>
<div id="ref-H1cK0KNl">
<p>16. <strong>mScarlet: a bright monomeric red fluorescent protein for cellular imaging</strong><br />
Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus WJ Gadella<br />
<em>Nature Methods</em> (2016-11-21) <a href="https://doi.org/10.1038/nmeth.4074" class="uri">https://doi.org/10.1038/nmeth.4074</a></p>
</div>
<div id="ref-oLbIsF1b">
<p>17. <strong>Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in theEscherichia coliinner membrane</strong><br />
Dorothee Kiefer, Xintong Hu, Ross Dalbey, Andreas Kuhn<br />
<em>The EMBO Journal</em> (1997-05-01) <a href="https://doi.org/10.1093/emboj/16.9.2197" class="uri">https://doi.org/10.1093/emboj/16.9.2197</a></p>
</div>
</div>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
