<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="João Silva" />
  <meta name="author" content="Soraia Lopes" />
  <meta name="author" content="Diogo Grilo" />
  <meta name="author" content="Zach Hensel" />
  <meta name="dcterms.date" content="2019-01-04" />
  <meta name="keywords" content="gene expression, synthetic biology, transcription" />
  <title>Plasmids for independently tunable, low-noise gene expression</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <link rel="stylesheet" href="github-pandoc.css">
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.2/MathJax.js?config=TeX-AMS_CHTML-full" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!-- Insert Analytics Script Below -->
  <script>
  </script>
  <!-- End Analytics Script -->
</head>
<body>
<header>
<h1 class="title">Plasmids for independently tunable, low-noise gene expression</h1>
</header>
<p><small><em> This manuscript (<a href="https://zach-hensel.github.io/low-noise-manuscript/v/2f592a0b8e2beed063df1b36895b19cb23f1e211/">permalink</a>) was automatically generated from <a href="https://github.com/zach-hensel/low-noise-manuscript/tree/2f592a0b8e2beed063df1b36895b19cb23f1e211">zach-hensel/low-noise-manuscript@2f592a0</a> on January 4, 2019. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>João Silva</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Soraia Lopes</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Diogo Grilo</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/XXXX-XXXX-XXXX-XXXX">XXXX-XXXX-XXXX-XXXX</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal </small></p></li>
<li><p><strong>Zach Hensel</strong><br> <img src="images/orcid.svg" alt="ORCID icon" width="13" height="13" /> <a href="https://orcid.org/0000-0002-4348-6229">0000-0002-4348-6229</a> · <img src="images/github.svg" alt="GitHub icon" width="13" height="13" /> <a href="https://github.com/zach-hensel">zach-hensel</a> · <img src="images/twitter.svg" alt="Twitter icon" width="13" height="13" /> <a href="https://twitter.com/alchemytoday">alchemytoday</a><br> <small> Instituto de Tecnologia Química e Biológica António Xavier, Universidade Nova de Lisboa, Oeiras, Portugal · Funded by Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) <br>Correspondence: <a href="mailto:zach.hensel@itqb.unl.pt">zach.hensel@itqb.unl.pt</a> </small></p></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Some microbiology experiments and biotechnology applications can be improved if it is possible to tune the expression of two different genes at the same time with cell-to-cell variation at or below the level of genes constitutively expressed from the chromosome (the “extrinsic noise limit”). This was recently achieved for a single gene by exploiting negative autoregulation by the tetracycline repressor (TetR) and bicistronic gene expression to reduce gene expression noise. We report new plasmids that use the same principles to achieve simultaneous, low-noise expression for two genes. The TetR system was moved to a compatible plasmid backbone, and a system based on the lactose repressor (LacI) was found to also exhibit gene expression noise below the extrinsic noise limit. We characterize gene expression mean and noise across the range of induction levels for these plasmids, apply the LacI system to tune expression for single-molecule mRNA detection in two different growth conditions, and show that two plasmids can be co-transformed to independently tune expression of two different genes.</p>
<h2 id="introduction" class="page_break_before">Introduction</h2>
<p>We recently reported the development of a plasmid-based gene-expression system in which a gene of interest is expressed bicistronically with the tetracycline repressor (TetR) <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span>. Using this gene expression system, cell-to-cell variation is below the “extrinsic noise limit” (coefficient of variation squared of protein concentration, <span class="math inline">\(CV^2 \approx 0.1\)</span>) observed for genes expressed from the chromosome <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>. When TetR and GFP are expressed bicistronically, GFP induction and gene expression noise is similar to that observed for a TetR-GFP fusion protein with autoregulation <span class="citation" data-cites="x1Hd2npA">[<a href="#ref-x1Hd2npA">3</a>]</span>. Compared to induction of gene expression under the control of a constitutively expressed transcriptional repressor, the inducer dose-response is relatively linearized, and gene expression noise is much lower at intermediate induction levels <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span>.</p>
<p>Our recent experiments in mRNA detection and other single-molecule experiments in living <em>E. coli</em> cells sometimes require the tunable expression of two different genes, both with low noise levels. For example, adopting a recently reported mRNA detection systems based on local enrichment of fluorescent RNA-binding proteins <span class="citation" data-cites="82diXnad">[<a href="#ref-82diXnad">4</a>]</span> for use in <em>E. coli</em> requires lower noise in protein production relative to the same system in <em>S. cerevisiae</em>, because of a smaller cell volume and the inability to sequester unbound protein in the nucleus. At the same time, tunable expression with low noise in the level of the target RNA is desired to make it possible to characterize the accuracy of RNA detection over a range of RNA levels. We hoped that expressing both the target RNA and RNA-binding fluorescent protein on two plasmids that could be tuned independently would simplify and accelerate development of new RNA-detection systems in <em>E. coli</em>. Achieving this was a three-step process: first, characterizing the TetR-based system on a compatible plasmid backbone; second, establishing an orthogonal, low-noise expression system based on the <em>lac</em> repressor (LacI); and third, showing that the two systems can be tuned independently.</p>
<h2 id="methods">Methods</h2>
<h3 id="strain-construction">Strain construction</h3>
<p>All plasmids were constructed using isothermal assembly <span class="citation" data-cites="dc6GAnhL">[<a href="#ref-dc6GAnhL">5</a>]</span> of fragments generated by PCR or double stranded DNA synthesis (IDT, Coralville) and transformed Top10 E. coli cells (5-1600-020, IBA Life Sciences, Göttingen). Transformants were screened by colony PCR and verified by sequencing. Purified plasmids were transformed into <em>E. coli</em> strain MG1655 by growing 3 mL of culture in SOB media at 30 °C to OD600=0.4, washing twice with 1 mL ice-cold water, resuspending in 40 µL water, electroporating 1-–10 ng plasmid with the EC1 setting of a Micropulser (Bio-Rad Laboratories, Hercules), and recovering for 1 hour at 37 °C in SOC media before plating on selective LB-agar.</p>
<p>To generate pJS101 with a compatible backbone, plasmid pZH509 <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span> was used as a template to amplify the bicistronic regulatory construct including the P<sub>LtetO-1</sub> promoter <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>, GFPmut2 <span class="citation" data-cites="13TJF2gcl">[<a href="#ref-13TJF2gcl">7</a>]</span>, tn10 TetR <span class="citation" data-cites="Q1a7tU8">[<a href="#ref-Q1a7tU8">8</a>]</span> and rrnB T1 transcription terminator <span class="citation" data-cites="xjKSoV9v">[<a href="#ref-xjKSoV9v">9</a>]</span>. This was assembled by isothermal assembly with the backbone from pGB2 <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">10</a>]</span> with the pSC101 origin of replication and spectinomycin resistance to generate plasmid pJS101. Plasmid pJS102 was generated by 3-fragment isothermal assembly. Plasmid pZH509 was used as a template both for the vector backbone and for GFPmut2, with non-homologous extensions added to PCR primers to generate the P<sub>LlacO-1</sub> promoter. LacI was amplified from <em>E. coli</em> MG1655 by colony PCR.</p>
<p>The test strain for mRNA imaging, ZHX99, was constructed similar to ZHX222 in recent work <span class="citation" data-cites="A1kNMUKB">[<a href="#ref-A1kNMUKB">11</a>]</span>. Similarly, in ZHX99 a construct in which a fusion protein of mVenus and Cro is expressed from the bacteriophage λ promoter <em>P<sub>R</sub></em> was integrated into the chromosome to replace the <em>lac</em> operon in MG1655 <span class="citation" data-cites="RBKXt78E">[<a href="#ref-RBKXt78E">12</a>]</span>. ZHX99 differs from ZHX222 in three ways. First, the <em>P<sub>R</sub></em> promoter was weakened by site-directed mutagenesis to produce a strain with lower mRNA levels. Second, a very strong ribosome binding site was added (RBS #136 <span class="citation" data-cites="pzTydEMA">[<a href="#ref-pzTydEMA">13</a>]</span>). Lastly, 24 tandem repeats of the recognition sequence for the PP7 coat protein were inserted between the open reading frame and transcription terminator (amplified by PCR from pDZ251 <span class="citation" data-cites="1GFLV3s4q">[<a href="#ref-1GFLV3s4q">14</a>]</span>). The pZH713 plasmid for mRNA detection was constructed by replacing GFPmut2 in pJS102 with a fusion protein of SYFP2 (amplified from a plasmid <span class="citation" data-cites="XuRR8Zmt">[<a href="#ref-XuRR8Zmt">15</a>]</span>) and the PP7 coat protein (PP7cp) (generated by codon optimization and DNA synthesis based on previously reported sequences <span class="citation" data-cites="EObeLKJ1">[<a href="#ref-EObeLKJ1">16</a>]</span>). Additionally, in pZH713 the PP7cp-SYFP2 fusion protein is translated from the weak ribosome binding site from pZH511 <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">17</a>]</span>.</p>
<p>To test independent induction of two genes, GFPmut2 in pJS102 was replaced by mScarlet-I (amplified from a plasmid <span class="citation" data-cites="zj8Ydkzo">[<a href="#ref-zj8Ydkzo">18</a>]</span>) to make pDG101. Plasmids were co-transformed into MG1655 by electroporation following the above protocol, except with 1 µL each undiluted plasmid (~20–-40 ng) and selecting on LB-agar plates with both spectinomycin and carbenicillin. Sequence maps are included in supplementary file and plasmids useful for constructing additionally 2-gene expression systems (pJS101 and pJS102, <strong>Table <a href="#tbl:plasmidList">1</a></strong>) are available at AddGene (deposit numbers 118280 and 118281) and have been verified by NGS <span class="citation" data-cites="q8xLcfYV">[<a href="#ref-q8xLcfYV">19</a>]</span>.</p>
<a name="tbl:plasmidList"></a>
<table>
<caption><span>Table 1:</span> Plasmids used in this study. </caption>
<thead>
<tr class="header">
<th>Plasmid</th>
<th>Ori</th>
<th>Promoter</th>
<th>GOI</th>
<th>Repressor Expression</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pZH501</td>
<td>p15a</td>
<td>CI-SNAP-tag</td>
<td><sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span></td>
</tr>
<tr class="even">
<td>pZH509</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td><span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span></td>
</tr>
<tr class="odd">
<td>pJS101</td>
<td>pSC101</td>
<td>GFPmut2</td>
<td>P<sub>LtetO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pJS102</td>
<td>p15a</td>
<td>GFPmut2</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="odd">
<td>pZH713</td>
<td>p15a</td>
<td>PP7cp-SYFP2</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
<tr class="even">
<td>pDG101</td>
<td>p15a</td>
<td>mScarlet-I</td>
<td>P<sub>LlacO-1</sub></td>
<td>bicistronic</td>
<td>This work</td>
</tr>
</tbody>
</table>
<h3 id="characterization-of-gfp-expression-by-flow-cytometry">Characterization of GFP expression by flow cytometry</h3>
<p>All flow cytometry experiment were repeated 3 times on different days and used plasmids transformed by electroporation into <em>E. coli</em> MG1655. Cultures were grown overnight at 30 °C from LB-agar plates supplemented with carbenicillin or spectinomycin (both at 50 <span class="math inline">\(\frac{\mu g}{mL}\)</span>) in 1 mL EZ Rich Defined Medium (M2105, Teknova, Hollister) supplemented with the same antibiotics. Cells were reinoculated 1:400 in 1 mL of the same media supplemented with Isopropyl β-D-1-thiogalactopyranoside (IPTG, at concentrations of 0, 2, 4.5, 10, 22.4, 50, 111.8, 250, 559, and 1250 µM) or anhydrotetrycline (ATc, at concentrations of 1, 5, 25, and 125 nM) as indicated and grown at 30 °C for 4–4.5 hours until reaching an optical density at 600 nm of 0.2–0.3. Next, 10 µL of cells were added to 1 mL of PBS at pH 7.4 and examined by flow cytometry.</p>
<p>Flow cytometry data was collected on an S3e cell sorter (Bio-Rad, Hercules) using a target flow rate of 2,000 counts/s and collecting 30,000 counts for each sample. A 488-nm laser line was used for excitation and its maximum power setting with amplification settings of 450 (forward scattering, FSC), 350 (side scattering, SSC), and 900 (FL1, 525/30 nm). The cell sorter is calibrated daily for a linear response to sample fluorescence intensity. Acquisition was triggered by forward scattering with a threshold of 3. Data was exported as an FCS file and imported into a custom Python script using FlowCal <span class="citation" data-cites="J6PWaBUu">[<a href="#ref-J6PWaBUu">20</a>]</span>. Following previous methods <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span>, one third of samples were selected based on proximity to the peak of FSC-AREA and SSC-HEIGHT in a 2D histogram using the density2d method in FlowCal. The FL1-AREA measurements were used to estimate the mean and variance of GFP distributions for all samples. This was done by estimating the probability density functions in bins distributed equally in logarithmic space and fitting by least squares minimization to a gamma function. We found that this method reduced the influence of low-FL1-AREA events that escaped other gating steps, and which had frequencies that varied for different samples and days (<strong>Figure <a href="#fig:gammaFits">S1</a></strong>). In all plots, the mean fluorescence of a strain harboring a similar plasmid, pZH501, that does not encode a fluorescent protein, was subtracted <span class="citation" data-cites="RB57dahw">[<a href="#ref-RB57dahw">1</a>]</span>.</p>
<p>We note that side scattering was weakly correlated with fluorescence, and thus with cell size, so this gating modestly reduces measured noise in FL1-AREA. However, we are comparing to an “extrinsic noise limit” determined from measurements of cell fluorescence divided by cell area <span class="citation" data-cites="oD6kpSs3">[<a href="#ref-oD6kpSs3">2</a>]</span>, which effectively does the same thing. In practice, we observed slightly lower noise measurements for GFP concentration in fluorescence microscopy images compared to total GFP fluorescence in the gated flow cytometry sample for similarly induced strains. This difference was more significant for very-low-expression conditions, and noise in conditions where GFP fluorescence distributions significantly overlap with ungated background events (GFP intensity less than 10<sup>4</sup> in <strong>Figure <a href="#fig:gammaFits">S1</a></strong>) is somewhat overestimated. The script for data analysis as well as all raw FCS data is available in <strong>Supplementary File S2</strong> and utilized modules from SciPy, NumPy, Matplotlib, and Pandas.</p>
<h3 id="microscopy">Microscopy</h3>
<p>For microscopy, cells were grown similarly to the protocol for flow cytometry. Differences for mRNA imaging were that cells were grown in M9 media supplemented with 1X MEM Amino Acids (M9A) or M9A additionally supplemented with 1% SOB media. Cells were imaged on agarose gel pads (3% BP165-25, Fisher Bio-Reagents) diluted in M9A (for mRNA imaging) or PBS (for two-plasmid imaging). For mRNA imaging, the microscope sample chamber was maintained at 30 °C. All imaging was done on a Leica DMI6000 inverted microscope using illumination from a Leica EL6000 source, fluorescence filter cubes (Leica GFP ET, a custom filter set with Semrock filters FF01-561/13, FF02-616/73, and DI02-R561, or the Semrock LF514-B filter set), a 100x/1.46 a-plan apochromat oil immersion objective, and an Evolve 512 EM-CCD camera (Photometrics). Images were prepared using Fiji <span class="citation" data-cites="1Xy8hvPg">[<a href="#ref-1Xy8hvPg">21</a>]</span>, with linear scaling and maintaining minimum and maximum intensity values for all comparable images.</p>
<h3 id="sm-fish">SM-FISH</h3>
<p>Cells were grown in overnight cultures at 32 °C in M9 minimal media supplemented with 1% SOB media, 50 µg/mL carbenicillin or 50 µg/mL spectinomycin. The next day, cells were diluted 1:100 or more (if multiple experiments were to be done sequentially over several hours) in the same media supplemented with anhydrotetracycline. In this growth condition, doubling time was approximately 30 minutes for MG1655 strains harboring these plasmids.</p>
<h3 id="microscopy-1">Microscopy</h3>
<p>Microscopy methods and microscopy data analysis methods</p>
<h3 id="flow-cytometry">Flow cytometry</h3>
<p>Flow cytometry methods and data analysis methods</p>
<h2 id="results">Results</h2>
<h3 id="moving-bicistronic-autoregulatory-construct-to-a-compatible-plasmid-backbone">Moving bicistronic autoregulatory construct to a compatible plasmid backbone</h3>
<p>The first step in creating a low-noise system for tuning expression of two genes is to establish that a previously characterized bicistronic autoregulatory circuit functions well in a compatible plasmid backbone. In this expression system, GFP and TetR are expressed bicistronically from the TetR-repressible promoter P<sub>LtetO-1</sub> and expression is induced by the addition of ATc <span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">17</a>]</span>. This system was shown to have low noise and a linearized dose response compared to a system in which TetR is constitutively expressed. We moved the system from a plasmid with a p15A replicon conferring ampicillin resistance to a lower-copy-number plasmid with a pSC101 replicon conferring spectinomycin resistance <span class="citation" data-cites="5CRbRLYC">[<a href="#ref-5CRbRLYC">10</a>]</span>. The p15A and pSC101 replicons have been used together in multiplasmid systems <span class="citation" data-cites="15TQ1MvYV">[<a href="#ref-15TQ1MvYV">22</a>]</span>.</p>
<p><span class="citation" data-cites="TlAItren">[<a href="#ref-TlAItren">17</a>]</span> my plosone paper <span class="citation" data-cites="15TQ1MvYV">[<a href="#ref-15TQ1MvYV">22</a>]</span> keasling biobricks</p>
<p>GFP expression mean and noise were characterized from low to high levels of induction by flow cytometry. <strong>Figure <a href="#fig:pZH509-pJS101">1</a></strong> shows that pJS101 induces at similar ATc concentrations as pZH509, with the change to the lower-copy pSC101 backbone resulting in a 58% drop in mean expression levels at a wide range of ATc concentrations. For a similar expression system in the absence of autoregulated TetR expression, moving the P<sub>LtetO-1</sub> promoter from a p15A to a pSC101 backbone resulted in an 87% drop in expression <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span>. A smaller change is expected in our experiment since negative autoregulation will provide dosage compensation, just autoregulation can reduce noise in plasmid copy number <span class="citation" data-cites="GcRoYBF2 Qz6zNiXK S7KRThVS">[<a href="#ref-GcRoYBF2">23</a>,<a href="#ref-Qz6zNiXK">24</a>,<a href="#ref-S7KRThVS">25</a>]</span>.</p>
<p><span class="citation" data-cites="GcRoYBF2">[<a href="#ref-GcRoYBF2">23</a>]</span> dublanche copy number variation <span class="citation" data-cites="Qz6zNiXK">[<a href="#ref-Qz6zNiXK">24</a>]</span> becskei serrano autoregulation noise <span class="citation" data-cites="S7KRThVS">[<a href="#ref-S7KRThVS">25</a>]</span> paulsson summing up the noise <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span> lutz laco1 etc</p>
<figure>
<img src="images/Fig1_pZH509_pJS101.png" alt="Figure 1: Influence of plasmid backbone on induction of TetR expression system. Mean single-cell GFP fluorescence for pZH509 (blue) and pJS101 (orange) plasmids as a function of ATc concentration. Error bars are 1 standard error." id="fig:pZH509-pJS101" style="width:89mm" /><figcaption><span>Figure 1:</span> <strong>Influence of plasmid backbone on induction of TetR expression system.</strong> Mean single-cell GFP fluorescence for pZH509 (blue) and pJS101 (orange) plasmids as a function of ATc concentration. Error bars are 1 standard error.</figcaption>
</figure>
<h3 id="alternative-regulatory-constructs-with-laci-replacing-tetr">Alternative regulatory constructs with LacI replacing TetR</h3>
<p>We hypothesized that replacing P<sub>LtetO-1</sub> with the inducible promoter P<sub>LlacO-1</sub> with similar characteristics <span class="citation" data-cites="NCWOk1rp">[<a href="#ref-NCWOk1rp">6</a>]</span> and replacing TetR with LacI might result in a similarly useful expression system that could be tuned independently. However, regulatory parameters for TetR and LacI vary wildly, with …</p>
<p>binding constants: tetR: LacI:</p>
<p>half induction: TetR ATc Tc LacI IPTG</p>
<p><strong>Figure <a href="#fig:pJS102-mean-noise">2</a></strong> shows</p>
<p>Note that MG1655 has the full <em>lac</em> operon, and that we have seen that the system works well in the presence and absence of the <em>lac</em> operon (MG1655/pJS102 vs ZHX99/pZH713)</p>
<figure>
<img src="images/Fig2_pJS102_mean_noise.png" alt="Figure 2: Characterizing mean expression levels and noise for different gene expression systems. (a) pJS102 mean induction (b) mean vs noise for all strains." id="fig:pJS102-mean-noise" style="width:183mm" /><figcaption><span>Figure 2:</span> <strong>Characterizing mean expression levels and noise for different gene expression systems.</strong> (<strong>a</strong>) pJS102 mean induction (<strong>b</strong>) mean vs noise for all strains.</figcaption>
</figure>
<h3 id="using-the-new-induction-system-for-detection-of-single-mrna-in-living-e.-coli">Using the new induction system for detection of single mRNA in living <em>E. coli</em></h3>
<p>blah blah</p>
<p><strong>Figure <a href="#fig:pZH713-optimization">3</a></strong> shows</p>
<figure>
<img src="images/Fig3_pZH713_optimization.png" alt="Figure 3: Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions. (a) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (b) Using the pJS102 expression system, an optimal range of 10–20 µM IPTG is quickly identified for single mRNA detection in minimal media." id="fig:pZH713-optimization" style="width:89mm" /><figcaption><span>Figure 3:</span> <strong>Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.</strong> (<strong>a</strong>) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (<strong>b</strong>) Using the pJS102 expression system, an optimal range of 10–20 µM IPTG is quickly identified for single mRNA detection in minimal media.</figcaption>
</figure>
<h3 id="independent-tunable-expression-of-two-genes">Independent, tunable expression of two genes</h3>
<p>Aggregation of mRNA-binding proteins from RNA bacteriophages makes it difficult to use fluorescent fusion proteins to detect single mRNAs <span class="citation" data-cites="cfHwILHp">[<a href="#ref-cfHwILHp">26</a>]</span>. We hypothesized that …</p>
<p><span class="citation" data-cites="ne2ewtkS">[<a href="#ref-ne2ewtkS">27</a>]</span> mNeonGreen <span class="citation" data-cites="H1cK0KNl">[<a href="#ref-H1cK0KNl">28</a>]</span> mScarlet-I <span class="citation" data-cites="oLbIsF1b">[<a href="#ref-oLbIsF1b">29</a>]</span> The Pf3 coat protein is a small membrane-spanning protein with a cytoplasm-facing C terminus.</p>
<p><strong>Figure <a href="#fig:pJS101-pDG101">4</a></strong> shows</p>
<figure>
<img src="images/Fig4_pJS101_pDG101.png" alt="Figure 4: Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions. (a) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (b) Using the pJS102 expression system, an optimal range of 10–20 µM IPTG is quickly identified for single mRNA detection in minimal media." id="fig:pJS101-pDG101" style="width:89mm" /><figcaption><span>Figure 4:</span> <strong>Using IPTG to tune expression of a fluorescent RNA-binding protein for single mRNA detection in different growth conditions.</strong> (<strong>a</strong>) 100 µM IPTG works well in rich media conditions, but has too high an expression level to see single mRNA spots above background in minimal media conditions. (<strong>b</strong>) Using the pJS102 expression system, an optimal range of 10–20 µM IPTG is quickly identified for single mRNA detection in minimal media.</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>We expect that the expression plasmids introduced here will be widely useful for applications in molecular biology. Adopting these systems for use with any</p>
<p>Possible to improve pJS23103 with somewhat stronger promoter?</p>
<p>Possible to extend to 3 genes? Possible using additional repressor, pMB1 origin of replication, and different antibiotic selection.</p>
<p>Why this would it be useful to do this for two different genes at the same time? Example for stoichiometric production of a protein complex Expressing genes at their native levels … e.g. for generating chromosome deletions</p>
<h2 id="author-contributions">Author contributions</h2>
<p>JS, DG and ZH designed experiments and performed experiments. JS, SL, DG, and ZH performed molecular cloning. ZH and JS analyzed data and wrote the paper. JS, SL, DG and ZH edited and approved the manuscript. ZH supervised the project.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>This work was financially supported by: Project LISBOA-01-0145-FEDER-007660 (Microbiologia Molecular, Estrutural e Celular) funded by FEDER funds through COMPETE2020—Programa Operacional Competitividade e Internacionalização (POCI), by national funds through FCT—Fundação para a Ciência e a Tecnologia, and through a joint research agreement with the Okinawa Institute of Science and Technology (OIST). This manuscript was composed and edited using Manubot <span class="citation" data-cites="1GGGHdsew">[<a href="#ref-1GGGHdsew">30</a>]</span>.</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs">
<div id="ref-RB57dahw">
<p>1. <strong>A plasmid-based Escherichia coli gene expression system with cell-to-cell variation below the extrinsic noise limit</strong><br />
Zach Hensel<br />
<em>PLOS ONE</em> (2017-10-30) <a href="https://doi.org/10.1371/journal.pone.0187259" class="uri">https://doi.org/10.1371/journal.pone.0187259</a></p>
</div>
<div id="ref-oD6kpSs3">
<p>2. <strong>Quantifying E. coli Proteome and Transcriptome with Single-Molecule Sensitivity in Single Cells</strong><br />
Y. Taniguchi, P. J. Choi, G.-W. Li, H. Chen, M. Babu, J. Hearn, A. Emili, X. S. Xie<br />
<em>Science</em> (2010-07-29) <a href="https://doi.org/10.1126/science.1188308" class="uri">https://doi.org/10.1126/science.1188308</a></p>
</div>
<div id="ref-x1Hd2npA">
<p>3. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular Systems Biology</em> (2006-08-01) <a href="https://doi.org/10.1038/msb4100081" class="uri">https://doi.org/10.1038/msb4100081</a></p>
</div>
<div id="ref-82diXnad">
<p>4. <strong>An improved MS2 system for accurate reporting of the mRNA life cycle.</strong><br />
Evelina Tutucci, Maria Vera, Jeetayu Biswas, Jennifer Garcia, Roy Parker, Robert H Singer<br />
<em>Nature methods</em> (2017-11-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29131164" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/29131164</a></p>
</div>
<div id="ref-dc6GAnhL">
<p>5. <strong>Enzymatic assembly of DNA molecules up to several hundred kilobases</strong><br />
Daniel G Gibson, Lei Young, Ray-Yuan Chuang, J Craig Venter, Clyde A Hutchison, Hamilton O Smith<br />
<em>Nature Methods</em> (2009-04-12) <a href="https://doi.org/10.1038/nmeth.1318" class="uri">https://doi.org/10.1038/nmeth.1318</a></p>
</div>
<div id="ref-NCWOk1rp">
<p>6. <strong>Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements.</strong><br />
R Lutz, H Bujard<br />
<em>Nucleic acids research</em> (1997-03-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/9092630</a></p>
</div>
<div id="ref-13TJF2gcl">
<p>7. <strong>FACS-optimized mutants of the green fluorescent protein (GFP).</strong><br />
BP Cormack, RH Valdivia, S Falkow<br />
<em>Gene</em> (1996) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8707053" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8707053</a></p>
</div>
<div id="ref-Q1a7tU8">
<p>8. <strong>Nucleotide sequence of the repressor gene of the TN10 tetracycline resistance determinant</strong><br />
Kathleen Postle, Toai T. Nguyen, Kevin P. Bertrand<br />
<em>Nucleic Acids Research</em> (1984) <a href="https://doi.org/10.1093/nar/12.12.4849" class="uri">https://doi.org/10.1093/nar/12.12.4849</a></p>
</div>
<div id="ref-xjKSoV9v">
<p>9. <strong>Intrinsic termination of T7 RNA polymerase mediated by either RNA or DNA.</strong><br />
L Hartvig, J Christiansen<br />
<em>The EMBO journal</em> (1996-09-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8887568" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/8887568</a></p>
</div>
<div id="ref-5CRbRLYC">
<p>10. <strong>A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.</strong><br />
G Churchward, D Belin, Y Nagamine<br />
<em>Gene</em> (1984-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6098521" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/6098521</a></p>
</div>
<div id="ref-A1kNMUKB">
<p>11. <strong>Cell-cycle-synchronized, oscillatory expression of a negatively autoregulated gene in E. coli</strong><br />
Zach Hensel, Tatiana T. Marquez-Lago<br />
<em>arXiv</em> (2015-06-29) <a href="https://arxiv.org/abs/1506.08596v1" class="uri">https://arxiv.org/abs/1506.08596v1</a></p>
</div>
<div id="ref-RBKXt78E">
<p>12. <strong>One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products.</strong><br />
KA Datsenko, BL Wanner<br />
<em>Proceedings of the National Academy of Sciences of the United States of America</em> (2000-06-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/10829079</a></p>
</div>
<div id="ref-pzTydEMA">
<p>13. <strong>Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.</strong><br />
Amin Espah Borujeni, Anirudh S Channarasappa, Howard M Salis<br />
<em>Nucleic acids research</em> (2013-11-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24234441" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/24234441</a></p>
</div>
<div id="ref-1GFLV3s4q">
<p>14. <strong>Real-time observation of transcription initiation and elongation on an endogenous yeast gene.</strong><br />
Daniel R Larson, Daniel Zenklusen, Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Science (New York, N.Y.)</em> (2011-04-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21512033" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/21512033</a></p>
</div>
<div id="ref-XuRR8Zmt">
<p>15. <strong>Cyan and yellow super fluorescent proteins with improved brightness, protein folding, and FRET Förster radius.</strong><br />
Gert-Jan Kremers, Joachim Goedhart, Erik B van Munster, Theodorus WJ Gadella<br />
<em>Biochemistry</em> (2006-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16716067" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16716067</a></p>
</div>
<div id="ref-EObeLKJ1">
<p>16. <strong>Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells.</strong><br />
Bin Wu, Jeffrey A Chao, Robert H Singer<br />
<em>Biophysical journal</em> (2012-06-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22735544" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/22735544</a></p>
</div>
<div id="ref-TlAItren">
<p>17. <strong>A plasmid-based Escherichia coli gene expression system with cell-to-cell variation below the extrinsic noise limit.</strong><br />
Zach Hensel<br />
<em>PloS one</em> (2017-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29084263" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/29084263</a></p>
</div>
<div id="ref-zj8Ydkzo">
<p>18. <strong>Characterization of a spectrally diverse set of fluorescent proteins as FRET acceptors for mTurquoise2.</strong><br />
Marieke Mastop, Daphne S Bindels, Nathan C Shaner, Marten Postma, Theodorus WJ Gadella, Joachim Goedhart<br />
<em>Scientific reports</em> (2017-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28931898" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/28931898</a></p>
</div>
<div id="ref-q8xLcfYV">
<p>19. <strong>Zach Hensel Lab Plasmids</strong><br />
Zach Hensel, João Silva<br />
<em>AddGene</em> (2019) <a href="https://www.addgene.org/Zach_Hensel/" class="uri">https://www.addgene.org/Zach_Hensel/</a></p>
</div>
<div id="ref-J6PWaBUu">
<p>20. <strong>FlowCal: A User-Friendly, Open Source Software Tool for Automatically Converting Flow Cytometry Data from Arbitrary to Calibrated Units.</strong><br />
Sebastian M Castillo-Hair, John T Sexton, Brian P Landry, Evan J Olson, Oleg A Igoshin, Jeffrey J Tabor<br />
<em>ACS synthetic biology</em> (2016-05-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27110723" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/27110723</a></p>
</div>
<div id="ref-1Xy8hvPg">
<p>21. <strong>Fiji: an open-source platform for biological-image analysis.</strong><br />
Johannes Schindelin, Ignacio Arganda-Carreras, Erwin Frise, Verena Kaynig, Mark Longair, Tobias Pietzsch, Stephan Preibisch, Curtis Rueden, Stephan Saalfeld, Benjamin Schmid, … Albert Cardona<br />
<em>Nature methods</em> (2012-06-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/22743772</a></p>
</div>
<div id="ref-15TQ1MvYV">
<p>22. <strong>BglBrick vectors and datasheets: A synthetic biology platform for gene expression.</strong><br />
Taek Soon Lee, Rachel A Krupa, Fuzhong Zhang, Meghdad Hajimorad, William J Holtz, Nilu Prasad, Sung Kuk Lee, Jay D Keasling<br />
<em>Journal of biological engineering</em> (2011-09-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21933410" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/21933410</a></p>
</div>
<div id="ref-GcRoYBF2">
<p>23. <strong>Noise in transcription negative feedback loops: simulation and experimental analysis.</strong><br />
Yann Dublanche, Konstantinos Michalodimitrakis, Nico Kümmerer, Mathilde Foglierini, Luis Serrano<br />
<em>Molecular systems biology</em> (2006-08-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16883354" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/16883354</a></p>
</div>
<div id="ref-Qz6zNiXK">
<p>24. <strong>Engineering stability in gene networks by autoregulation.</strong><br />
A Becskei, L Serrano<br />
<em>Nature</em> (2000-06-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10850721" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/10850721</a></p>
</div>
<div id="ref-S7KRThVS">
<p>25. <strong>Summing up the noise in gene networks.</strong><br />
Johan Paulsson<br />
<em>Nature</em> (2004-01-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14749823" class="uri">https://www.ncbi.nlm.nih.gov/pubmed/14749823</a></p>
</div>
<div id="ref-cfHwILHp">
<p>26. <strong>Spatial organization of the flow of genetic information in bacteria</strong><br />
Paula Montero Llopis, Audrey F. Jackson, Oleksii Sliusarenko, Ivan Surovtsev, Jennifer Heinritz, Thierry Emonet, Christine Jacobs-Wagner<br />
<em>Nature</em> (2010-06-20) <a href="https://doi.org/10.1038/nature09152" class="uri">https://doi.org/10.1038/nature09152</a></p>
</div>
<div id="ref-ne2ewtkS">
<p>27. <strong>A bright monomeric green fluorescent protein derived from Branchiostoma lanceolatum</strong><br />
Nathan C Shaner, Gerard G Lambert, Andrew Chammas, Yuhui Ni, Paula J Cranfill, Michelle A Baird, Brittney R Sell, John R Allen, Richard N Day, Maria Israelsson, … Jiwu Wang<br />
<em>Nature Methods</em> (2013-03-24) <a href="https://doi.org/10.1038/nmeth.2413" class="uri">https://doi.org/10.1038/nmeth.2413</a></p>
</div>
<div id="ref-H1cK0KNl">
<p>28. <strong>mScarlet: a bright monomeric red fluorescent protein for cellular imaging</strong><br />
Daphne S Bindels, Lindsay Haarbosch, Laura van Weeren, Marten Postma, Katrin E Wiese, Marieke Mastop, Sylvain Aumonier, Guillaume Gotthard, Antoine Royant, Mark A Hink, Theodorus WJ Gadella<br />
<em>Nature Methods</em> (2016-11-21) <a href="https://doi.org/10.1038/nmeth.4074" class="uri">https://doi.org/10.1038/nmeth.4074</a></p>
</div>
<div id="ref-oLbIsF1b">
<p>29. <strong>Negatively charged amino acid residues play an active role in orienting the Sec-independent Pf3 coat protein in theEscherichia coliinner membrane</strong><br />
Dorothee Kiefer, Xintong Hu, Ross Dalbey, Andreas Kuhn<br />
<em>The EMBO Journal</em> (1997-05-01) <a href="https://doi.org/10.1093/emboj/16.9.2197" class="uri">https://doi.org/10.1093/emboj/16.9.2197</a></p>
</div>
<div id="ref-1GGGHdsew">
<p>30. <strong>Open collaborative writing with Manubot</strong><br />
Daniel S. Himmelstein, David R. Slochower, Venkat S. Malladi, Casey S. Greene, Anthony Gitter<br />
<em>Manubot Preprint</em> (2018) <a href="https://greenelab.github.io/meta-review/" class="uri">https://greenelab.github.io/meta-review/</a></p>
</div>
</div>
<h2 id="supplementary-material" class="page_break_before">Supplementary Material</h2>
<figure>
<img src="images/FigureS1_gammaFits.png" alt="Figure S1: Reproducibility of low-noise expression in 3 independent experiments. Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples." id="fig:gammaFits" data-tag="S1" style="width:89mm" /><figcaption><span>Figure S1:</span> <strong>Reproducibility of low-noise expression in 3 independent experiments.</strong> Probability densities for each flow cytometry sample were calculated by kernel density estimates for the negative control plasmid pZH501 (orange), ZH509 (blue), pJS101 (green), and pJS102 (red) with fluorescence levels monotonically increasing with concentration of ATc (1, 5, 25, 125 nM) or IPTG (0, 2, 4.5, 10, 22.5, 50, 111.8, 250, 559, 1250). Distributions were fit by least squares regression to a gamma function (black dashed lines) to estimate sample mean and variance while minimizing the influence of non-fluorescent background events, which varied in frequency for different days and samples.</figcaption>
</figure>
<p><strong>Supplementary File S2</strong>: Raw flow cytometry data, Python scripts required to reproduce Figures <a href="#fig:pZH509-pJS101">1</a> and <a href="#fig:pJS102-mean-noise">2</a>, DNA sequences, and explanatory text files are available as a compressed archive at Zenodo.</p>
<script>
// AnchorJS minified version below.
// Source https://github.com/bryanbraun/anchorjs/blob/064abdd0987f305933ec4982af6d0c1cf2fd0814/anchor.js

/**
 * AnchorJS - v4.0.0 - 2017-06-02
 * https://github.com/bryanbraun/anchorjs
 * Copyright (c) 2017 Bryan Braun; Licensed MIT
 */
!function(A,e){"use strict";"function"==typeof define&&define.amd?define([],e):"object"==typeof module&&module.exports?module.exports=e():(A.AnchorJS=e(),A.anchors=new A.AnchorJS)}(this,function(){"use strict";function A(A){function e(A){A.icon=A.hasOwnProperty("icon")?A.icon:"",A.visible=A.hasOwnProperty("visible")?A.visible:"hover",A.placement=A.hasOwnProperty("placement")?A.placement:"right",A.class=A.hasOwnProperty("class")?A.class:"",A.truncate=A.hasOwnProperty("truncate")?Math.floor(A.truncate):64}function t(A){var e;if("string"==typeof A||A instanceof String)e=[].slice.call(document.querySelectorAll(A));else{if(!(Array.isArray(A)||A instanceof NodeList))throw new Error("The selector provided to AnchorJS was invalid.");e=[].slice.call(A)}return e}function n(){if(null===document.head.querySelector("style.anchorjs")){var A,e=document.createElement("style");e.className="anchorjs",e.appendChild(document.createTextNode("")),void 0===(A=document.head.querySelector('[rel="stylesheet"], style'))?document.head.appendChild(e):document.head.insertBefore(e,A),e.sheet.insertRule(" .anchorjs-link {   opacity: 0;   text-decoration: none;   -webkit-font-smoothing: antialiased;   -moz-osx-font-smoothing: grayscale; }",e.sheet.cssRules.length),e.sheet.insertRule(" *:hover > .anchorjs-link, .anchorjs-link:focus  {   opacity: 1; }",e.sheet.cssRules.length),e.sheet.insertRule(" [data-anchorjs-icon]::after {   content: attr(data-anchorjs-icon); }",e.sheet.cssRules.length),e.sheet.insertRule(' @font-face {   font-family: "anchorjs-icons";   src: url(data:n/a;base64,AAEAAAALAIAAAwAwT1MvMg8yG2cAAAE4AAAAYGNtYXDp3gC3AAABpAAAAExnYXNwAAAAEAAAA9wAAAAIZ2x5ZlQCcfwAAAH4AAABCGhlYWQHFvHyAAAAvAAAADZoaGVhBnACFwAAAPQAAAAkaG10eASAADEAAAGYAAAADGxvY2EACACEAAAB8AAAAAhtYXhwAAYAVwAAARgAAAAgbmFtZQGOH9cAAAMAAAAAunBvc3QAAwAAAAADvAAAACAAAQAAAAEAAHzE2p9fDzz1AAkEAAAAAADRecUWAAAAANQA6R8AAAAAAoACwAAAAAgAAgAAAAAAAAABAAADwP/AAAACgAAA/9MCrQABAAAAAAAAAAAAAAAAAAAAAwABAAAAAwBVAAIAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAMCQAGQAAUAAAKZAswAAACPApkCzAAAAesAMwEJAAAAAAAAAAAAAAAAAAAAARAAAAAAAAAAAAAAAAAAAAAAQAAg//0DwP/AAEADwABAAAAAAQAAAAAAAAAAAAAAIAAAAAAAAAIAAAACgAAxAAAAAwAAAAMAAAAcAAEAAwAAABwAAwABAAAAHAAEADAAAAAIAAgAAgAAACDpy//9//8AAAAg6cv//f///+EWNwADAAEAAAAAAAAAAAAAAAAACACEAAEAAAAAAAAAAAAAAAAxAAACAAQARAKAAsAAKwBUAAABIiYnJjQ3NzY2MzIWFxYUBwcGIicmNDc3NjQnJiYjIgYHBwYUFxYUBwYGIwciJicmNDc3NjIXFhQHBwYUFxYWMzI2Nzc2NCcmNDc2MhcWFAcHBgYjARQGDAUtLXoWOR8fORYtLTgKGwoKCjgaGg0gEhIgDXoaGgkJBQwHdR85Fi0tOAobCgoKOBoaDSASEiANehoaCQkKGwotLXoWOR8BMwUFLYEuehYXFxYugC44CQkKGwo4GkoaDQ0NDXoaShoKGwoFBe8XFi6ALjgJCQobCjgaShoNDQ0NehpKGgobCgoKLYEuehYXAAAADACWAAEAAAAAAAEACAAAAAEAAAAAAAIAAwAIAAEAAAAAAAMACAAAAAEAAAAAAAQACAAAAAEAAAAAAAUAAQALAAEAAAAAAAYACAAAAAMAAQQJAAEAEAAMAAMAAQQJAAIABgAcAAMAAQQJAAMAEAAMAAMAAQQJAAQAEAAMAAMAAQQJAAUAAgAiAAMAAQQJAAYAEAAMYW5jaG9yanM0MDBAAGEAbgBjAGgAbwByAGoAcwA0ADAAMABAAAAAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAP) format("truetype"); }',e.sheet.cssRules.length)}}this.options=A||{},this.elements=[],e(this.options),this.isTouchDevice=function(){return!!("ontouchstart"in window||window.DocumentTouch&&document instanceof DocumentTouch)},this.add=function(A){var i,o,s,c,r,a,h,l,u,d,f,g,p=[];if(e(this.options),"touch"===(g=this.options.visible)&&(g=this.isTouchDevice()?"always":"hover"),A||(A="h2, h3, h4, h5, h6"),0===(i=t(A)).length)return this;for(n(),o=document.querySelectorAll("[id]"),s=[].map.call(o,function(A){return A.id}),r=0;r<i.length;r++)if(this.hasAnchorJSLink(i[r]))p.push(r);else{if(i[r].hasAttribute("id"))c=i[r].getAttribute("id");else if(i[r].hasAttribute("data-anchor-id"))c=i[r].getAttribute("data-anchor-id");else{u=l=this.urlify(i[r].textContent),h=0;do{void 0!==a&&(u=l+"-"+h),a=s.indexOf(u),h+=1}while(-1!==a);a=void 0,s.push(u),i[r].setAttribute("id",u),c=u}d=c.replace(/-/g," "),(f=document.createElement("a")).className="anchorjs-link "+this.options.class,f.href="#"+c,f.setAttribute("aria-label","Anchor link for: "+d),f.setAttribute("data-anchorjs-icon",this.options.icon),"always"===g&&(f.style.opacity="1"),""===this.options.icon&&(f.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(f.style.lineHeight="inherit")),"left"===this.options.placement?(f.style.position="absolute",f.style.marginLeft="-1em",f.style.paddingRight="0.5em",i[r].insertBefore(f,i[r].firstChild)):(f.style.paddingLeft="0.375em",i[r].appendChild(f))}for(r=0;r<p.length;r++)i.splice(p[r]-r,1);return this.elements=this.elements.concat(i),this},this.remove=function(A){for(var e,n,i=t(A),o=0;o<i.length;o++)(n=i[o].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[o]))&&this.elements.splice(e,1),i[o].removeChild(n));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){var t=/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\]/g;return this.options.truncate||e(this.options),A.trim().replace(/\'/gi,"").replace(t,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&(" "+A.firstChild.className+" ").indexOf(" anchorjs-link ")>-1,t=A.lastChild&&(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ")>-1;return e||t||!1}}return A});

// Enable links for selected headers
var anchors = new AnchorJS();
anchors.add("h2, h3, h4")
</script>
<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
<script src="https://hypothes.is/embed.js" async></script>
</body>
</html>
